<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>darwin_core</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="darwin_core_files/libs/clipboard/clipboard.min.js"></script>
<script src="darwin_core_files/libs/quarto-html/quarto.js"></script>
<script src="darwin_core_files/libs/quarto-html/popper.min.js"></script>
<script src="darwin_core_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="darwin_core_files/libs/quarto-html/anchor.min.js"></script>
<link href="darwin_core_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="darwin_core_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="darwin_core_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="darwin_core_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="darwin_core_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">



<section id="darwin-core" class="level3">
<h3 class="anchored" data-anchor-id="darwin-core">Darwin Core</h3>
<p><strong>Contents</strong></p>
<ul>
<li><a href="#introduction-to-darwin-core">Introduction to Darwin Core</a></li>
<li><a href="#darwin-core-terms">Darwin Core terms</a><br>
</li>
<li><a href="#darwin-core-guidelines">Darwin Core guidelines</a>
<ul>
<li><a href="#taxonomy-and-identification">Taxonomy and identification</a></li>
<li><a href="#occurrence">Occurrence</a><br>
</li>
<li><a href="#record-level-terms">Record level terms</a><br>
</li>
<li><a href="#location">Location</a><br>
</li>
<li><a href="#event">Event</a></li>
<li><a href="#time">Time</a><br>
</li>
<li><a href="#sampling">Sampling</a></li>
</ul></li>
</ul>
<section id="introduction-to-darwin-core" class="level4">
<h4 class="anchored" data-anchor-id="introduction-to-darwin-core">Introduction to Darwin Core</h4>
<p><a href="https://dwc.tdwg.org/">Darwin Core</a> is a body of standards (i.e., identifiers, labels, definitions) that facilitate sharing biodiversity informatics. It provides stable <a href="https://dwc.tdwg.org/terms/">terms</a> and vocabularies related to biological objects/data and their collection. Darwin Core is maintained by <a href="http://tdwg.org/">TDWG (Biodiversity Information Standards, formerly The International Working Group on Taxonomic Databases)</a>. Stable terms and vocabularies are important for ensuring the datasets in OBIS have consistently interpretable fields. By following Darwin Core standards, both data providers and users can be certain of the definition and quality of data.</p>
<section id="history-of-darwin-core-and-obis" class="level5">
<h5 class="anchored" data-anchor-id="history-of-darwin-core-and-obis">History of Darwin Core and OBIS</h5>
<p>The old <a href="http://old.iobis.org/node/304">OBIS schema</a> was an <a href="http://old.iobis.org/obis/obis.xsd">OBIS extension</a> to Darwin Core 1.2., which was based on <a href="http://rs.tdwg.org/dwc/terms/simple/">Simple Darwin Core</a>, a subset of Darwin Core which does not allow any structure beyond rows and columns. This old schema added some terms which were important for OBIS, but were not supported by Darwin Core at the time (e.g., start and end date and start and end latitude and longitude, depth range, lifestage, and terms for abundance, biomass and sample size).</p>
<p>In 2009, the Executive Committee of TDWG announced their ratification of an updated version of Darwin Core as a <a href="https://www.tdwg.org/community/dwc/#history-and-context">TDWG Standard</a>. Ratified Darwin Core unifies specializations and innovations emerging from diverse communities, and provides guidelines for ongoing enhancement. The <a href="https://dwc.tdwg.org/terms/">Darwin Core Quick Reference Guide</a> links to TDWG’s term definitions and related practices for Ratified Darwin Core. We will discuss the relevance of terms in this guide further below.</p>
<p>In December 2013, the <a href="https://obis.org/about/">3rd session of the IODE Steering Group for OBIS</a> agreed to transition OBIS globally to the TDWG-Ratified version of Darwin Core, and the mapping of the (old) OBIS specific terms to Darwin Core can be found <a href="http://rs.tdwg.org/dwc/terms/history/versions/index.htm#dwcobis">here</a>.</p>
<p><a class="anchor" name="terms" href=""></a></p>
</section>
</section>
<section id="darwin-core-dwc-terms" class="level4">
<h4 class="anchored" data-anchor-id="darwin-core-dwc-terms">Darwin Core (DwC) terms</h4>
<p>DwC terms correspond to the column names of your dataset and can be grouped according to class type for convenience, e.g., Taxa, Occurrence, Record, Location, etc. It is important to use DwC field names because only columns using Darwin Core terms as headers will be recognized.</p>
<p>A list of all possible Darwin Core terms can be found on <a href="https://dwc.tdwg.org/terms/">TDWG</a>. However, OBIS does not parse all terms (note this doesn’t mean you cannot include them, they just will not be parsed when you publish to OBIS). Below is an overview of the most relevant Darwin Core terms to consider when contributing to OBIS, with guidelines regarding their use. We have also compiled a convenient <a href="checklist.html">checklist</a> of OBIS-accepted terms, their DwC class type, and which OBIS file (Event Core, Occurrence, eMoF, etc.) it is likely to be found in.</p>
<p>Note that OBIS currently has seven required and one strongly recommended DwC term: <code>occurrenceID</code>, <code>eventDate</code>, <code>decimalLongitude</code>, <code>decimalLatitude</code>, <code>scientificName</code>, <code>occurrenceStatus</code>, <code>basisOfRecord</code>, <code>scientificNameID</code> (strongly recommended).</p>
<p>The following DwC terms are related to the Class <em>Taxon</em>:</p>
<ul>
<li>scientificName</li>
<li>scientificNameID</li>
<li>scientificNameAuthorship</li>
<li>kingdom</li>
<li>taxonRank</li>
<li>taxonRemarks</li>
</ul>
<p>The following DwC terms are related to the Class <em>Identification</em>:</p>
<ul>
<li>identifiedBy</li>
<li>dateIdentified</li>
<li>identificationReferences</li>
<li>identificationRemarks</li>
<li>identificationQualifier</li>
<li>typeStatus</li>
</ul>
<p>The following DwC terms are related to the Class <em>Occurrence</em>:</p>
<ul>
<li>occurrenceID</li>
<li>occurrenceStatus</li>
<li>recordedBy</li>
<li>individualCount (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>organismQuantity (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>organismQuantityType (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>sex (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>lifeStage (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>behavior</li>
<li>associatedTaxa</li>
<li>occurrenceRemarks</li>
<li>associatedMedia</li>
<li>associatedReferences</li>
<li>associatedSequences</li>
<li>catalogNumber</li>
<li>preparations</li>
</ul>
<p>The following DwC terms are related to the Class <em>Record level</em>:</p>
<ul>
<li>basisOfRecord</li>
<li>institutionCode</li>
<li>collectionCode</li>
<li>collectionID</li>
<li>bibliographicCitation</li>
<li>modified</li>
<li>dataGeneralizations</li>
</ul>
<p>The following DwC terms are related to the Class <em>Location</em>:</p>
<ul>
<li>decimalLatitude</li>
<li>decimalLongitude</li>
<li>coordinateUncertaintyInMeters</li>
<li>geodeticDatum</li>
<li>footprintWKT</li>
<li>minimumDepthInMeters</li>
<li>maximumDepthInMeters</li>
<li>minimumDistanceAboveSurfaceInMeters</li>
<li>maximumDistanceAboveSurfaceInMeters</li>
<li>locality</li>
<li>waterBody</li>
<li>islandGroup</li>
<li>island</li>
<li>country</li>
<li>locationAccordingTo</li>
<li>locationRemarks</li>
<li>locationID</li>
</ul>
<p>The following DwC terms are related to the Class <em>Event</em>:</p>
<ul>
<li>parentEventID</li>
<li>eventID</li>
<li>eventDate</li>
<li>type</li>
<li>habitat</li>
<li>samplingProtocol (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>sampleSizeValue (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>SampleSizeUnit (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>samplingEffort (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
</ul>
<p>The following DwC terms are related to the Class <em>MaterialSample</em>:</p>
<ul>
<li>materialSampleID</li>
</ul>
</section>
<section id="darwin-core-guidelines" class="level4">
<h4 class="anchored" data-anchor-id="darwin-core-guidelines">Darwin Core guidelines</h4>
<section id="taxonomy-and-identification" class="level5">
<h5 class="anchored" data-anchor-id="taxonomy-and-identification">Taxonomy and identification</h5>
<p><code>scientificName</code> (required term) should always contain the originally recorded scientific name, even if the name is currently a synonym. This is necessary to be able to track back records to the original dataset. The name should be at the lowest possible taxonomic rank, preferably at species level or lower, but higher ranks, such as genus, family, order, class etc. are also acceptable. We recommend to not include authorship in <code>scientificName</code>, and only use <code>scientificNameAuthorship</code> for that purpose. The <code>scientificName</code> term should only contain the name and not identification qualifications (such as ?, confer or affinity), which should instead be supplied in the <code>IdentificationQualifier</code> term, see examples below. <code>taxonRemarks</code> can capture comments or notes about the taxon or name.</p>
<p>A <a href="http://www.marinespecies.org/">WoRMS</a> LSID should be added in <code>scientificNameID</code> (strongly recommended term), OBIS will use this identifier to pull the taxonomic information from the World Register of Marine Species (WoRMS) into OBIS and attach it to your dataset. This information includes:</p>
<ul>
<li>Taxonomic classification (kingdom through species)</li>
<li>The accepted name in case of invalid names or synonyms</li>
<li>AphiaID</li>
<li>IUCN red list category</li>
</ul>
<p>LSIDs are persistent, location-independent, resource identifiers for uniquely naming biologically significant resources. More information on LSIDs can be found at <a href="http://www.lsid.info/">www.lsid.info</a>. For example, the WoRMS LSID for <em>Solea solea</em> is: urn:lsid:marinespecies.org:taxname:127160, and can be found at the bottom of each WoRMS taxon page, e.g.&nbsp;<a href="http://marinespecies.org/aphia.php?p=taxdetails&amp;id=127160"><em>Solea solea</em></a>.</p>
<p><code>kingdom</code> and <code>taxonRank</code> can help us in identifying the provided <code>scientificName</code> in case the name is not available in WoRMS. <code>kingdom</code> in particular can help us find alternative genus-species combinations and avoids linking the name to homonyms. Please contact the WoRMS data management team (info@marinespecies.org) in case the scientificName is missing in WoRMS. <code>kingdom</code> and <code>taxonRank</code> are not necessary when a correct <code>scientificNameID</code> is provided.</p>
<p>OBIS recommends providing information about how an identification was made, for example by which ID key, species guide or expert; and by which method (e.g morphology vs.&nbsp;genomics), etc. The person’s name who made the taxonomic identification can go in <code>identifiedBy</code> and <em>when</em> in <code>dateIdentified</code>. Use the ISO 8601:2004(E) standard for date and time, for instructions see <a href="#time">Time</a>. A list of references, such as field guides used for the identification can be listed in <code>identificationReferences</code>. Any other information, such as identification methods, can be added to <code>identificationRemarks</code>.</p>
<p>Examples:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption></caption>
<colgroup>
<col style="width: 43%">
<col style="width: 23%">
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 12%">
</colgroup>
<thead>
<tr class="header">
<th>scientificNameID</th>
<th>scientificName</th>
<th>kingdom</th>
<th>phylum</th>
<th>class</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>urn:lsid:marinespecies.org:taxname:142004</td>
<td>Yoldiella nana</td>
<td>Animalia</td>
<td>Mollusca</td>
<td>Bivalvia</td>
</tr>
<tr class="even">
<td>urn:lsid:marinespecies.org:taxname:140584</td>
<td>Ennucula tenuis</td>
<td>Animalia</td>
<td>Mollusca</td>
<td>Bivalvia</td>
</tr>
<tr class="odd">
<td>urn:lsid:marinespecies.org:taxname:131573</td>
<td>Terebellides stroemii</td>
<td>Animalia</td>
<td>Annelida</td>
<td>Polychaeta</td>
</tr>
</tbody>
</table>
</div>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption><em>Data from <a href="http://ipt.vliz.be/eurobis/resource?r=largenet_k2">Benthic fauna around Franz Josef Land</a>.</em></caption>
<thead>
<tr class="header">
<th>order</th>
<th>family</th>
<th>genus</th>
<th>specificEpithet</th>
<th>scientificNameAuthorship</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Nuculanoida</td>
<td>Yoldiidae</td>
<td>Yoldiella</td>
<td>nana</td>
<td>(Sars M., 1865)</td>
</tr>
<tr class="even">
<td>Nuculoida</td>
<td>Nuculidae</td>
<td>Ennucula</td>
<td>tenuis</td>
<td>(Montagu, 1808)</td>
</tr>
<tr class="odd">
<td>Terebellida</td>
<td>Trichobranchidae</td>
<td>Terebellides</td>
<td>stroemii</td>
<td>Sars, 1835</td>
</tr>
</tbody>
</table>
</div>
<p>If the record represents a nomenclatural type specimen, the term <code>typeStatus</code> can be used, e.g.&nbsp;for holotype, syntype, etc.</p>
<p><strong>In case of low confidence identifications</strong>, and the scientific name contains qualifiers such as <em>cf.</em>, <em>?</em> or <em>aff.</em>, then this name should go in <code>identificationQualifier</code>, and <code>scientificName</code> should contain the name of the lowest possible taxon rank that refers to the most accurate identification. E.g. if the specimen was accurately identified down to genus level, but not species level, then the scientificName should contain the name of the genus, the scientificNameID should contain the LSID the genus and the <code>identificationQualifier</code> should contain the low confidence species name combined with <em>?</em> or other qualifiers. The table below shows a few examples:</p>
<p>The use and definitions for additional ON signs (<code>identificationQualifier</code>) can be found in <a href="https://doi.org/10.1111/2041-210X.12594">Open Nomenclature in the biodiversity era</a>, which provides examples for using the main Open Nomenclature qualifiers associated with <em>physical specimens</em>. The publication <a href="https://www.frontiersin.org/articles/10.3389/fmars.2021.620702/full">Recommendations for the Standardisation of Open Taxonomic Nomenclature for Image-Based Identiﬁcations</a> provides examples and definitions for <code>identificationQualifiers</code> for <em>non-physical specimens (image-based)</em>.</p>
<p>Examples:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption></caption>
<colgroup>
<col style="width: 11%">
<col style="width: 22%">
<col style="width: 24%">
<col style="width: 6%">
<col style="width: 14%">
<col style="width: 20%">
</colgroup>
<thead>
<tr class="header">
<th>scientificName</th>
<th>scientificNameAuthorship</th>
<th>scientificNameID</th>
<th>taxonRank</th>
<th>identificationQualifier</th>
<th>taxonConceptID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Pelagia</td>
<td>Péron &amp; Lesueur, 1810</td>
<td>urn:lsid:marinespecies.org:taxname:135262</td>
<td>genus</td>
<td>gen. nov.</td>
<td>Pelagia gen. nov.</td>
</tr>
<tr class="even">
<td>Pelagia benovici</td>
<td>Piraino, Aglieri, Scorrano &amp; Boero, 2014</td>
<td>urn:lsid:marinespecies.org:taxname:851656</td>
<td>species</td>
<td>sp. nov</td>
<td>Pelagia benovici sp. nov</td>
</tr>
<tr class="odd">
<td>Gadus</td>
<td>Linnaeus, 1758</td>
<td>urn:lsid:marinespecies.org:taxname:125732</td>
<td>genus</td>
<td>cf.&nbsp;morhua</td>
<td>Gadus cf.&nbsp;morhua</td>
</tr>
<tr class="even">
<td>Polycera</td>
<td>Cuvier, 1816</td>
<td>urn:lsid:marinespecies.org:taxname:138369</td>
<td>genus</td>
<td>cf.&nbsp;hedgpethi</td>
<td>Polycera cf.&nbsp;hedgpethi</td>
</tr>
<tr class="odd">
<td>Tubifex</td>
<td>Lamarck, 1816</td>
<td>urn:lsid:marinespecies.org:taxname:137392</td>
<td>genus</td>
<td>?</td>
<td>Tubifex tubifex(Müller, 1774)?</td>
</tr>
<tr class="even">
<td>Tubifex</td>
<td>Lamarck, 1816</td>
<td>urn:lsid:marinespecies.org:taxname:137392</td>
<td>genus</td>
<td>sp. inc.</td>
<td>Tubifex tubifex(Müller, 1774)sp. inc.</td>
</tr>
<tr class="odd">
<td>Brisinga</td>
<td>Asbjørnsen, 1856</td>
<td>urn:lsid:marinespecies.org:taxname:123210</td>
<td>genus</td>
<td>gen. inc.</td>
<td>Brisinga gen. inc.</td>
</tr>
<tr class="even">
<td>Uroptychus compressus</td>
<td>Baba &amp; Wicksten, 2019</td>
<td>urn:lsid:marinespecies.org:taxname:1332465</td>
<td>genus</td>
<td>sp. inc.</td>
<td>Uroptychus compressus sp. inc.</td>
</tr>
<tr class="odd">
<td>Eurythenes</td>
<td>S. I. Smith in Scudder, 1882</td>
<td>urn:lsid:marinespecies.org:taxname:101607</td>
<td>genus</td>
<td>sp. DISCOLL.PAP.JC165.674</td>
<td>Eurythenes sp.DISCOLL.PAP.JC165.674</td>
</tr>
<tr class="even">
<td>Paroriza</td>
<td>Hérouard, 1902</td>
<td>urn:lsid:marinespecies.org:taxname:123467</td>
<td>genus</td>
<td>sp.[unique123]aff.pallens</td>
<td>Paroriza sp.[unique123]aff. pallens</td>
</tr>
<tr class="odd">
<td>Aristeidae</td>
<td>Wood-Mason in Wood-Mason &amp; Alcock, 1891</td>
<td>urn:lsid:marinespecies.org:taxname:106725</td>
<td>family</td>
<td>stet.</td>
<td>Aristeidae stet.</td>
</tr>
<tr class="even">
<td>Nematocarcinus</td>
<td>Milne-Edwards, 1881</td>
<td>urn:lsid:marinespecies.org:taxname:107015</td>
<td>genus</td>
<td>sp.indet.</td>
<td>Nematocarcinus sp.indet.</td>
</tr>
<tr class="odd">
<td>Brisinga</td>
<td>Asbjørnsen, 1856</td>
<td>urn:lsid:marinespecies.org:taxname:123210</td>
<td>genus</td>
<td>gen.inc.</td>
<td>Brisinga gen.inc.</td>
</tr>
<tr class="even">
<td>Brisinga costata</td>
<td>Verrill, 1884</td>
<td>urn:lsid:marinespecies.org:taxname:17825</td>
<td>species</td>
<td>sp.inc.</td>
<td>Brisinga costata sp.inc.</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="occurrence" class="level5">
<h5 class="anchored" data-anchor-id="occurrence">Occurrence</h5>
<p><code>occurrenceID</code> (required term) is an identifier for the occurrence record and should be persistent and globally unique. If the dataset does not yet contain (globally unique) occurrenceIDs, then they should be created. Guideline for ID creation can be found <a href="identifiers.html">here</a></p>
<p><code>occurrenceStatus</code> (required term) is a statement about the presence or absence of a taxon at a location. It is an important term, because it allows us to distinguish between presence and absence records. It is a required term and should be filled in with either <code>present</code> or <code>absent</code>.</p>
<p>A few terms related to quantity: <code>organismQuantity</code> and <code>organismQuantityType</code>, have been added to the TDWG ratified Darwin Core. This is a lot more versatile than the older <code>individualCount</code> field. However, OBIS recommends to use the <a href="data_format.html#extendedmeasurementorfact-extension-emof">Extended MeasurementorFact extension</a> for quantitative measurements because of the standardization of terms and the fact that you can link these measurements to sampling events and factual sampling information.</p>
<p>Please take note that OBIS recommends all quantitative measurements and sampling facts to be placed in the <code>ExtendedMeasurementOrFact</code> extension and not in the Darwin Core files.</p>
<p>In the case specimens were collected and stored (e.g.&nbsp;museum collections), the <code>catalogNumber</code> and <code>preparations</code> terms can be used to provide the identifier for the record in the collection and to document the preparation and preservation methods. The term <code>typeStatus</code> see above (under identification) can be used in this context too.</p>
<p>Both <code>associatedMedia</code>, <code>associatedReferences</code> and <code>associatedSequences</code> are global unique identifiers or URIs pointing to respectively associated media (e.g.&nbsp;online image or video), associated literature (e.g.&nbsp;DOIs) or genetic sequence information (e.g.&nbsp;GenBANK ID).</p>
<p><code>associatedTaxa</code> include a list (concatenated and separated) of identifiers or names of taxa and their associations with the Occurrence, e.g.&nbsp;the species occurrence was associated to the presence of kelp such as <em>Laminaria digitata</em>.</p>
<p>The recommended vocabulary for <code>sex</code> see <a href="http://vocab.nerc.ac.uk/collection/S10/current/">BODC vocab : S10</a>, for <code>lifeStage</code> see <a href="http://vocab.nerc.ac.uk/collection/S11/current/">BODC vocab: S11</a>, <code>behavior</code> (no vocab available), and <code>occurrenceRemarks</code> can hold any comments or notes about the Occurrence.</p>
<p><code>recordedBy</code> can hold a list (concatenated and separated) of names of people, groups, or organizations responsible for recording the original Occurrence. The primary collector or observer, especially one who applies a personal identifier (recordNumber), should be listed first.</p>
<p>Example:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption><em>Data from <a href="http://ipt.vliz.be/eurobis/resource?r=summary_of_benthic_studies_in_the_sluice_dock_ostend_during_1976-1981">A summary of benthic studies in the sluice dock of Ostend during 1976-1981</a>.</em></caption>
<thead>
<tr class="header">
<th>collectionCode</th>
<th>occurrenceID</th>
<th>catalogNumber</th>
<th>occurrenceStatus</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SluiceDock_benthic_1976/1981</td>
<td>SluiceDock_benthic_1976_1</td>
<td>SluiceDock_benthic_1976_1</td>
<td>present</td>
</tr>
<tr class="even">
<td>SluiceDock_benthic_1976/1981</td>
<td>SluiceDock_benthic_1976_2</td>
<td>SluiceDock_benthic_1976_2</td>
<td>present</td>
</tr>
<tr class="odd">
<td>SluiceDock_benthic_1976/1981</td>
<td>SluiceDock_benthic_1979-07/1980-06_1</td>
<td>SluiceDock_benthic_1979-07/1980-06_1</td>
<td>present</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="record-level-terms" class="level5">
<h5 class="anchored" data-anchor-id="record-level-terms">Record level terms</h5>
<p><code>basisOfRecord</code> (required term) specifies the nature of the record, i.e.&nbsp;whether the occurrence record is based on a stored specimen or an observation. In case the specimen is collected and stored in a collection (e.g.&nbsp;at a museum, university, research institute), the options are:</p>
<ul>
<li><code>PreservedSpecimen</code> e.g.&nbsp;preserved in ethanol, tissue etc.</li>
<li><code>FossilSpecimen</code> a fossil, which allows OBIS to make the distinction between the date of collection and the time period the specimen was assumed alive</li>
<li><code>LivingSpecimen</code> an intentionally kept/cultivated living specimen e.g.&nbsp;in an aquarium or culture collection.</li>
</ul>
<p>In case no specimen is deposited, the basis of record is either <code>HumanObservation</code> (e.g bird sighting, benthic sample but specimens were discarded after counting), or <code>MachineObservation</code> (e.g.&nbsp;for occurrences based on automated sensors such as image recognition, etc). For records pertaining to genetic samples, basisOfRecord can be <code>MaterialSample</code> (e.g.&nbsp;in the DNA-derived data extension).</p>
<p>When the basisOfRecord is either a <em>preservedSpecimen</em>, <em>LivingSpecimen</em> or <em>FossilSpecimen</em> please also add the <code>institutionCode</code>, <code>collectionCode</code> and <code>catalogNumber</code>, which will enable people to visit the collection and re-examine the material. Sometimes, for example in case of living specimens, a dataset can contain records pointing to the origin, the in-situ sampling position as well as a record referring to the ex-situ collection. In this case please add the event type information in <code>eventRemarks</code> (see <a href="darwin_core.html#event">OBIS manual: event</a>).</p>
<p><code>institutionCode</code> identifies the custodian institute (often by acronym), <code>collectionCode</code> identifies the collection or dataset within that institute. Collections cannot belong to multiple institutes, so all records within a collection should have the same <code>institutionCode</code>. The <code>collectionID</code> is an identifier for the record within the dataset or collection.</p>
<p><code>bibliographicCitation</code> allows for providing different citations on record level, while a single citation for the entire dataset can and should be provided in the metadata (see <a href="eml.html">EML</a>). The citation at record level can have the format of a chapter in a book, where the book is the dataset citation. The record citation will have preference over the dataset citation. We do not, however, recommend to create different citations for every record, as this will explode the number of citations and will hamper the re-use of data.</p>
<p><code>modified</code> is the most recent date-time on which the resource was changed. It is required to use the ISO 8601:2004(E) standard, for instructions see <a href="#time">Time</a>.</p>
<p><code>dataGeneralizations</code> refers to actions taken to make the shared data less specific or complete than in its original form. Suggests that alternative data of higher quality may be available on request. This can be the case for occurrences of vulnerable or endangered species and there positions are converted to the center of grid cells.</p>
<p><a class="anchor" name="location" href=""></a></p>
</section>
<section id="location" class="level5">
<h5 class="anchored" data-anchor-id="location">Location</h5>
<p><code>decimalLatitude</code> and <code>decimalLongitude</code> (required terms) are the geographic latitude and longitude (in decimal degrees), using the spatial reference system given in <code>geodeticDatum</code> of the geographic center of a Location. The number of decimals should be appropriate for the level of uncertainty in <code>coordinateUncertaintyInMeters</code> (at least within an order of magnitude). <code>coordinateUncertaintyInMeters</code> is the radius of the smallest circle around the given position containing the whole location. Regarding <code>decimalLatitude</code>, positive values are north of the Equator, negative values are south of it. All values lie between -90 and 90, inclusive. Regarding <code>decimalLongitude</code>, positive values are east of the Greenwich Meridian, negative values are west of it. All values lie between -180 and 180, inclusive.</p>
<p>In OBIS, the spatial reference system to be documented in <code>geodeticDatum</code> is <a href="https://epsg.io/4326">EPSG:4326</a>. Coordinates in degrees/minutes/seconds can be converted to decimal degrees using our <a href="http://iobis.github.io/coordinates/">coordinates tool</a>. We also provide a <a href="https://obis.org/maptool">tool</a> to check coordinates or to determine coordinates for a location (point, transect or polygon) on a map. This tool also allows geocoding location names using <a href="http://www.marineregions.org/">marineregions.org</a>.</p>
<p>The name of the place or location can be provided in <code>locality</code>, and if possible linked by a <code>locationID</code> using a persistent ID from a gazetter, such as the MRGID from <a href="http://www.marineregions.org/gazetteer.php?p=search">MarineRegions</a>. If the species occurrence only contains the name of the <code>locality</code>, but not the exact coordinates, we recommend using a geocoding service to obtain the coordinates. <a href="http://www.marineregions.org/">Marine Regions</a> has a <a href="http://www.marineregions.org/gazetteer.php?p=search">search interface</a> for geographic names, and provides coordinates and often precision in meters, which can go into <code>coordinateUncertaintyInMeters</code>. Another option is to use the <a href="http://www.getty.edu/research/tools/vocabularies/tgn/">Getty Thesaurus of Geographic Names</a> or <a href="http://maps.google.com">Google Maps</a>: after looking up a location, the decimal coordinates can be found in the page URL. Additional information about the locality can also be stored in DwC terms such as <code>waterBody</code>, <code>islandGroup</code>, <code>island</code> and <code>country</code>. <code>locationAccordingTo</code> should provide the name of the gazetteer that is used to obtain the coordinates for the locality.</p>
<p><code>locationID</code> is an identifier for the set of location information (e.g.&nbsp;station ID, or MRGID from <a href="http://www.marineregion.org">marineregions</a>), for example the <a href="http://www.marineregions.org/gazetteer.php?p=details&amp;id=3956">Balearic Plain</a> has MRGID: <a href="http://marineregions.org/mrgid/3956">http://marineregions.org/mrgid/3956</a>.</p>
<p>A <a href="https://en.wikipedia.org/wiki/Well-known_text">Well-Known Text</a> (WKT) representation of the shape of the location can be provided in <code>footprintWKT</code>. This is particularly useful for tracks, transects, tows, trawls, habitat extent or when an exact location is not known. WKT strings can be created using our <a href="https://obis.org/maptool">WKT tool</a>. This tool also calculates a midpoint and a radius, which can then be added to <code>decimalLongitude</code>, <code>decimalLatitude</code>, and <code>coordinateUncertaintyInMeters</code> respectively. There is also an <a href="https://github.com/iobis/obistools#calculate-centroid-and-radius-for-wkt-geometries">R tool</a> to calculate the centroid and radius for WKT polygons. <a href="https://wktmap.com">wktmap.com</a> can be used to visualize and share WKT strings.</p>
<!--<img src="images/wkttool2.png" class="img-responsive"/>-->
<p><img src="images/wkttool2.png" class="img-fluid"></p>
<p>Some examples of WKT strings:</p>
<pre class="text"><code>LINESTRING (30 10, 10 30, 40 40)
POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))
MULTILINESTRING ((10 10, 20 20, 10 40),(40 40, 30 30, 40 20, 30 10))
MULTIPOLYGON (((30 20, 45 40, 10 40, 30 20)),((15 5, 40 10, 10 20, 5 10, 15 5)))</code></pre>
<p>Example:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption><em>Data from <a href="http://ipt.vliz.be/eurobis/resource?r=ionian_2008_2018">Adriatic and Ionian Sea mega-fauna monitoring employing ferry as platform of observation along the Ancona-Igoumenitsa-Patras lane, from December 2014 to December 2018</a>.</em></caption>
<thead>
<tr class="header">
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>geodeticDatum</th>
<th>coordinateUncertaintyInMeters</th>
<th>footprintWKT</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>38.698</td>
<td>20.95</td>
<td>EPSG:4326</td>
<td>75033.17</td>
<td>LINESTRING (20.31 39.15, 21.58 38.24)</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>42.72</td>
<td>15.228</td>
<td>EPSG:4326</td>
<td>154338.87</td>
<td>LINESTRING (16.64 41.80, 13.82 43.64)</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>39.292</td>
<td>20.364</td>
<td>EPSG:4326</td>
<td>162083.27</td>
<td>LINESTRING (19.05 40.34, 21.68 38.25)</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
</div>
<p>Keep in mind while filling in <a href="http://rs.tdwg.org/dwc/terms/maximumDepthInMeters"><code>minimumDepthInMeters</code></a> and <a href="http://rs.tdwg.org/dwc/terms/minimumDepthInMeters"><code>maximumDepthInMeters</code></a> that this should be the depth at which the <strong>sample was taken</strong> and not the water column depth at that location. When fillling in any depth fields (<code>minimumDepthInMeters</code>, <code>maximumDepthInMeters</code>, <a href="http://rs.tdwg.org/dwc/terms/minimumDistanceAboveSurfaceInMeters"><code>minimumDistanceAboveSurfaceInMeters</code></a>, and <a href="http://rs.tdwg.org/dwc/terms/maximumDistanceAboveSurfaceInMeters"><code>maximumDistanceAboveSurfaceInMeters</code></a>), you should also consider which information is needed to fully understand the data. In most cases (e.g.&nbsp;scenario 1 and 4 in the figure below), providing <code>minimumDepthInMeters</code> and <code>maximumDepthInMeters</code> is sufficient for observations of organisms at particular depths. However, in cases where an occurrence is above the sea surface, e.g.&nbsp;flying birds (scenario 2 and 5), you should populate <code>minimumDistanceAboveSurfaceInMeters</code>, <code>maximumDistanceAboveSurfaceInMeters</code>, and, where relevant, you should also include <a href="https://dwc.tdwg.org/list/#dwc_minimumElevationInMeters"><code>minimumElevationInMeters</code></a> and <a href="https://dwc.tdwg.org/list/#dwc_maximumElevationInMeters"><code>maximumElevationInMeters</code></a>.</p>
<p>The <code>minimumDistanceAboveSurfaceInMeters</code> and <code>maximumDistanceAboveSurfaceInMeters</code> is the distance, in meters, above or below a reference surface or reference point. The reference surface is determined by the depth or elevation. If the depth and elevation are 0, then the reference surface is the sea surface. If a depth is given, the reference surface is the location of the depth. This can be especially useful for sediment cores taken from the sea bottom (scenario 3 in figure below). If no depth is given, then the elevation is the reference surface (scenario 5).</p>
<p><img src="images/Depth-figure-updated.png" class="img-fluid"></p>
<p>Depth scenario examples:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption></caption>
<colgroup>
<col style="width: 6%">
<col style="width: 13%">
<col style="width: 13%">
<col style="width: 22%">
<col style="width: 22%">
<col style="width: 10%">
<col style="width: 10%">
</colgroup>
<thead>
<tr class="header">
<th>Scenario</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
<th>minimumDistanceAboveSurfaceInMeters</th>
<th>maximumDistanceAboveSurfaceInMeters</th>
<th>minimumElevationInMeters</th>
<th>maximumElevationInMeters</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>40, 90</td>
<td>50, 100</td>
<td>-</td>
<td>-</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="even">
<td>2</td>
<td>0</td>
<td>0</td>
<td>10</td>
<td>15</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="odd">
<td>3</td>
<td>100</td>
<td>100</td>
<td>0</td>
<td>-1.5</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="even">
<td>4</td>
<td>20</td>
<td>22</td>
<td>-</td>
<td>-</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="odd">
<td>5</td>
<td>0</td>
<td>0</td>
<td>10</td>
<td>15</td>
<td>10</td>
<td>10</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="event" class="level5">
<h5 class="anchored" data-anchor-id="event">Event</h5>
<p><code>eventID</code> is an identifier for the sampling or observation event. <code>parentEventID</code> is an identifier for a parent event, which is composed of one or more sub-sampling (child) events (eventIDs). See <a href="identifiers.html#eventid">identifiers</a> for details on how these terms can be constructed.</p>
<p><code>habitat</code> is a category or description of the habitat in which the Event occurred (e.g.&nbsp;benthos, seamount, hydrothermal vent, seagrass, rocky shore, intertidal, ship wreck etc.)</p>
</section>
<section id="time" class="level5">
<h5 class="anchored" data-anchor-id="time">Time</h5>
<p>The date and time at which an occurrence was recorded goes in <code>eventDate</code>. This term uses the <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601 standard</a> and OBIS recommends using the extended ISO 8601 format with hyphens.</p>
<!--<img src="https://imgs.xkcd.com/comics/iso_8601.png" class="img-responsive"/>-->
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://imgs.xkcd.com/comics/iso_8601.png" class="img-fluid figure-img"></p>
</figure>
</div>
<p>More specific guidelines on formatting dates and times can be found in the <a href="common_formatissues.html#temporal-dates-and-times">Common Data formatting issues page</a></p>
</section>
<section id="sampling" class="level5">
<h5 class="anchored" data-anchor-id="sampling">Sampling</h5>
<p>Information on <code>sampleSizeValue</code> and <code>sampleSizeUnit</code> is very important when an organism quantity is specified. However, with <a href="data_format.html">OBIS-ENV-DATA</a> it was felt that the extended MeasurementorFact (<a href="http://rs.gbif.org/extension/obis/extended_measurement_or_fact.xml">eMoF</a>) extension would be better suited than the DwC Event Core to store the sampled area and/or volume because in some cases sampleSize by itself may not be detailed enough to allow interpretation of the sample. For instance, in the case of a plankton tow, the volume of water that passed through the net is relevant. In case of Niskin bottles, the volume of sieved water is more relevant than the actual volume in the bottle. In these examples, as well as generally when recording sampling effort for all protocols, eMoF enables greater flexibility to define parameters, as well as the ability to describe the entire sample and treatment protocol through multiple parameters. eMoF also allows you to standardize your terms to a controlled vocabulary.</p>
<p>The next chapter deals with the metadata (description of the dataset) in <a href="eml.html">Ecological Metadata Language</a>.</p>
</section>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>