<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>OBIS Manual - 2&nbsp; What can you contribute and how?</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./formatting.html" rel="next">
<link href="./intro.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar floating nav-fixed slimcontent">


<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    <img src="./logo.png" alt="OBIS" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">OBIS Manual</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link active" href="./index.html" rel="" target="" aria-current="page">
 <span class="menu-text">Manual</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./tutorials.html" rel="" target="">
 <span class="menu-text">Tutorials</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./packages.html" rel="" target="">
 <span class="menu-text">Packages</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./find-dwc.html" rel="" target="">
 <span class="menu-text">Find your DwC</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./contribute.html">Contributing data to OBIS</a></li><li class="breadcrumb-item"><a href="./contribute.html"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">What can you contribute and how?</span></a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Making better use of OBIS data</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Overview</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./intro.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
 <span class="menu-text">Contributing data to OBIS</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./contribute.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">What can you contribute and how?</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true">
 <span class="menu-text">Data Formatting</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./formatting.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Dataset structure</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./checklist.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Formatting data tables</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true">
 <span class="menu-text">Ensuring Data Quality</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./data_qc.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Data quality control</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" aria-expanded="true">
 <span class="menu-text">Publishing Data</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./data_publication.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Data publication and sharing</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true">
 <span class="menu-text">Access Data from OBIS</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./access.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Data access</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" aria-expanded="true">
 <span class="menu-text">Data Visualization and Analysis</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataviz.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Data Visualization</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" aria-expanded="true">
 <span class="menu-text">Additional Resources</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./other_resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Other Resources</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#why-publish-data-to-obis" id="toc-why-publish-data-to-obis" class="nav-link active" data-scroll-target="#why-publish-data-to-obis"><span class="header-section-number">2.1</span> Why publish data to OBIS</a></li>
  <li><a href="#how-to-handle-sensitive-data" id="toc-how-to-handle-sensitive-data" class="nav-link" data-scroll-target="#how-to-handle-sensitive-data"><span class="header-section-number">2.2</span> How to handle sensitive data</a></li>
  <li><a href="#obis-data-life-cycle" id="toc-obis-data-life-cycle" class="nav-link" data-scroll-target="#obis-data-life-cycle"><span class="header-section-number">2.3</span> OBIS Data Life Cycle</a></li>
  <li><a href="#biodiversity-data-standards" id="toc-biodiversity-data-standards" class="nav-link" data-scroll-target="#biodiversity-data-standards"><span class="header-section-number">2.4</span> Biodiversity data standards</a>
  <ul class="collapse">
  <li><a href="#darwin-core" id="toc-darwin-core" class="nav-link" data-scroll-target="#darwin-core"><span class="header-section-number">2.4.1</span> Darwin Core</a></li>
  <li><a href="#darwin-core-archive" id="toc-darwin-core-archive" class="nav-link" data-scroll-target="#darwin-core-archive"><span class="header-section-number">2.4.2</span> Darwin Core Archive</a></li>
  <li><a href="#relational-databases-the-underlying-framework-of-obis" id="toc-relational-databases-the-underlying-framework-of-obis" class="nav-link" data-scroll-target="#relational-databases-the-underlying-framework-of-obis"><span class="header-section-number">2.4.3</span> Relational databases: the underlying framework of OBIS</a></li>
  <li><a href="#ecological-metadata-language" id="toc-ecological-metadata-language" class="nav-link" data-scroll-target="#ecological-metadata-language"><span class="header-section-number">2.4.4</span> Ecological Metadata Language</a></li>
  </ul></li>
  <li><a href="#obis-nodes" id="toc-obis-nodes" class="nav-link" data-scroll-target="#obis-nodes"><span class="header-section-number">2.5</span> OBIS nodes</a>
  <ul class="collapse">
  <li><a href="#terms-of-reference-of-obis-nodes" id="toc-terms-of-reference-of-obis-nodes" class="nav-link" data-scroll-target="#terms-of-reference-of-obis-nodes"><span class="header-section-number">2.5.1</span> Terms of Reference of OBIS nodes</a></li>
  <li><a href="#how-to-become-an-obis-node" id="toc-how-to-become-an-obis-node" class="nav-link" data-scroll-target="#how-to-become-an-obis-node"><span class="header-section-number">2.5.2</span> How to become an OBIS node</a></li>
  <li><a href="#obis-node-health-status-check-and-transition-strategy" id="toc-obis-node-health-status-check-and-transition-strategy" class="nav-link" data-scroll-target="#obis-node-health-status-check-and-transition-strategy"><span class="header-section-number">2.5.3</span> OBIS Node Health Status Check and Transition Strategy</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">What can you contribute and how?</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<p>Since 2000, OBIS has accepted, curated and published marine biodiversity data obtained by varied sources and methods. There is a common misconception that OBIS only accepts species occurrence data - however this is not true! OBIS can accept many types of marine data including:</p>
<ul>
<li>Presence/Absence</li>
<li>Abundance, individual count</li>
<li>Biomass</li>
<li>Abiotic measurements</li>
<li>Biotic measurements</li>
<li>Sampling methods</li>
<li>Sample processing methods</li>
<li>Genetic data including sequences</li>
<li>Data originating from <a href="common_formatissues.html">historical records</a></li>
<li>Tracking data</li>
<li>Habitat data</li>
<li>Acoustic data</li>
<li>Imaging data</li>
<li>Metadata describing the dataset and any project or programme related metadata</li>
</ul>
<p>So if you have any of these types of marine data linked to your occurrence data and also want to contribute to OBIS - great! OBIS accepts data from any organization, consortium, project or individual who wants to contribute data. OBIS Data Sources are the authors, editors, and/or organisations that have published one or more datasets through OBIS. They remain the owners or custodians of the data, not OBIS!</p>
<p>OBIS harvests and publishes data from recognized IPTs from OBIS nodes or GBIF publishers. If you own data or have the right to publish data in OBIS, you can contact the <a href="https://obis.org/contact/">OBIS secretariat or one of the OBIS nodes</a>, or additionally a GBIF publisher. Your organization or programme can also <a href="nodes.html">become an OBIS node</a>. An OBIS node usually publishes data from multiple data holders, effectively being a node in a network of data providers. So you may have to first find a <a href="https://obis.org/contact/">relevant node</a> before you get your data ready to publish.</p>
<p>To publish a dataset to OBIS, there are <strong>five</strong> main steps you must go through.</p>
<ol type="1">
<li>First, you must <a href="https://obis.org/contact/">identify</a> which OBIS node is best suited to host your published data. If you would like to <a href="data_sharing.html#simultaneous-publishing-to-gbif">publish to GBIF</a> at the same time, that is also possible. If your organization is already affiliated with a GBIF node with which you must publish from, OBIS can also <a href="data_sharing.html#simultaneous-publishing-to-gbif">harvest from GBIF nodes</a>.</li>
<li>Second, you must determine the <a href="formatting.html">structure</a> of your data and which format will best suit your dataset. OBIS follows Darwin Core Archive (DwC-A) standards for datasets, and currently follows a star schema format. This format is based on relational databases. If you are unfamiliar with such database structures, or would like to refamiliarize yourself with them, please read <a href="relational_db.html">here</a></li>
<li>Then, you need to actually <a href="formatting.html">format</a> your data according to OBIS and DwC-A standards and guidelines</li>
<li>Once formatted, you should run a series of <a href="data_qc.html">quality control</a> measures to ensure you are not missing any required information and that all standards are being met. This helps ensure all data published in OBIS is formatted in a standardized way. When published in OBIS, OBIS provides a quality report to inform data owners and users of any quality control issues. By completing quality control before you publish your dataset you ensure there are fewer errors to fix later.</li>
<li>Now that your dataset is ready for publishing, the relevant metadata must be filled in, and then published on the previously identified IPT.</li>
</ol>
<p>Each of these steps are covered in detail in the relevant sections of the manual. For an overview of this process see <a href="data_standards.html">data management flow in OBIS</a>.</p>
<section id="why-publish-data-to-obis" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="why-publish-data-to-obis"><span class="header-section-number">2.1</span> Why publish data to OBIS</h2>
<p>It is important to publish and ensure your dataset follows a universal standard for several reasons. The <a href="https://www.nature.com/articles/sdata201618">FAIR guiding principles</a> for scientific data management and stewardship provide a good framework to understand the reasoning behind publishing data. FAIR stands for Findable, Accessible, Interoperable, and Reusable. Let’s understand each aspect within the FAIR framework and how it is linked to publishing data in OBIS.</p>
<ul>
<li><strong>F - Findable</strong></li>
</ul>
<p>Even if you publish your dataset on its own, publishing your data with OBIS will make your data more Findable (and Accessible) to a wider audience you might not have otherwise reached. By publishing your dataset to OBIS you are adding to a global database where your data can be found and analyzed alongside thousands of other datasets. For example, a dataset on <a href="https://obis.org/dataset/98ea98e8-356b-4dd5-87c6-4eeb7e3cf588">marine invasive species in Venezuela</a> was published July 20, 2022 and as of October 5, 2022 records of this dataset were included in 1,873 data download requests. This can save you time rather than handling individual data requests.</p>
<ul>
<li><strong>A - Accessible</strong></li>
</ul>
<p>Similar to being Findable, OBIS makes your datasets more Accessible. Each dataset is given an identifier when you upload it on an IPT. Thus when users obtain data from OBIS, the original dataset can easily be identified and accessed. Data from OBIS is accessible in <a href="access.html">numerous ways</a>, giving data users multiple avenues to potentially access your data.</p>
<ul>
<li><strong>I - Interoperable</strong></li>
</ul>
<p>Using a standardized data format with controlled vocabularies will ensure your data are more Interoperable - more easily interpreted and processed by computers and humans alike. Increasingly, scientists use computer programs to conduct e-Science and collect data with algorithms. Formatting your data for OBIS will ensure it can be read and accessed by such programs as well as understood by users.</p>
<ul>
<li><strong>R - Reusable</strong></li>
</ul>
<p>Publishing your data allows it to be Reused according to your chosen <a href="policy.html">data usage license</a>. Very likely you expended resources to collect your data and it would be a waste of those resources to leave your unique data unpublished and inaccessible for current and future generations. Likewise, it is better to preserve any data processing done to ensure your dataset is reproducible and/or verifiable. Finally, data in OBIS is often used in several assessment processes and used as information to support policy makers around the globe making informed decisions.</p>
<p>There are many other benefits of publishing in OBIS, even if you haven’t published any work on it yet. This includes:</p>
<ul>
<li>Your dataset can be <a href="data_sharing#adding-a-doi-to-datasets">associated with a DOI</a>, allowing for your dataset to be more easily cited. By ensuring your dataset citation is complete you will ensure you are being cited properly.</li>
<li>Publishing your dataset with OBIS makes it easier to set it up as a <a href="https://www.gbif.org/data-papers">Data paper</a>, which generates value for you and other researchers.</li>
<li>There are social benefits to data publishing as your work becomes integrated into a wider dataset. It gives both you and your data more visibility. This can lead to more opportunities for collaboration and further career development as a researcher or professional.</li>
<li>Your data can be incorporated into larger analyses to better understand global ocean biodiversity, helping to shape regional and international policies.</li>
</ul>
</section>
<section id="how-to-handle-sensitive-data" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="how-to-handle-sensitive-data"><span class="header-section-number">2.2</span> How to handle sensitive data</h2>
<p>We recognize that sometimes your dataset may contain sensitive information (e.g., location data on endangered or poached species), or perhaps your organization does not want certain details publicly accessible. Types of sensitive data include:</p>
<ul>
<li>Location data on endangered or protected species</li>
<li>Information regarding a commonly poached species</li>
<li>Species or locations that have an economic impact (positive or negative)</li>
</ul>
<p>To accommodate sensitivity but still be able to contribute to OBIS, we suggest:</p>
<ul>
<li><a href="common_qc.html#uncertain-geolocation">Generalizing location</a> information by: Obtaining regional coordinates using <a href="http://www.marineregions.org/gazetteer.php?p=search">MarineRegions</a>, <a href="http://www.getty.edu/research/tools/vocabularies/tgn/">Getty Thesaurus of Geographic Names</a>, or <a href="http://maps.google.com/">Google Maps</a></li>
<li>Using the <a href="https://obis.org/maptool/">OBIS Map tool</a> to generate a polygon area with a Well-Known Text (WKT) representation of the geometry to paste into the <code>footprintWKT</code> field.</li>
<li>Delay timing of publication (e.g., to accommodate mobile species)</li>
<li><a href="ipt.html">Submit your dataset, but mark it as private in the IPT</a> so it is not published right away (i.e., until you set it as public). Alternatively, you can set a password on your dataset in order to share with specific individuals. Note that setting passwords will require some coordination with the IPT manager. By submitting your data to an IPT but not immediately publishing it, you can ensure that the dataset will be in a place to be incorporated at a later date when it is ready to be made public. This not only saves time and helps retain details while relatively fresh in your mind, but also ensures the dataset is still ready to be mobilized in case jobs are changed at a later date.</li>
</ul>
<p>GBIF has created the following <a href="https://docs.gbif.org/sensitive-species-best-practices/master/en/">Best Practices for Generalizing Sensitive data</a> which can provide you with additional guidance. <code>Chapman AD (2020) Current Best Practices for Generalizing Sensitive Species Occurrence Data. Copenhagen: GBIF Secretariat. https://doi.org/10.15468/doc-5jp4-5g10.</code></p>
</section>
<section id="obis-data-life-cycle" class="level2 page-columns page-full" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="obis-data-life-cycle"><span class="header-section-number">2.3</span> OBIS Data Life Cycle</h2>
<!--<img align="right" src="images/OBISdataLifeCycle.png" class="img-responsive" style="margin:50px 0px" display: inline-block/>-->

<div class="no-row-height column-margin column-container"><div class="">
<p><img src="images/OBISdataLifeCycle.png" class="img-fluid"></p>
</div></div><p>The basic data life cycle for contributions to OBIS can be broken down into six step-by-step phases:</p>
<ol type="1">
<li>Data structure</li>
<li>Data formatting</li>
<li>Quality control</li>
<li>Publishing</li>
<li>Data access (downloading)</li>
<li>Data visualization</li>
</ol>
<p>Each of these phases are outlined in this manual and are composed of a number of steps which are covered in the relevant sections.</p>
<p>After you have decided on your <a href="formatting.html">data structure</a> and have moved to the Data Formatting stage, you must first <a href="name_matching.html">match</a> the taxa in your dataset to a registered list. In formatting your dataset you will ensure the <a href="checklist.html">required OBIS terms</a> and <a href="identifiers.html">identifiers</a> are mapped correctly to your data fields and records.</p>
<p>Depending on your data structure, you will then format data into a <a href="data_format.html">DwC-A</a> format with the appropriate Core table (<a href="format_event.html">Event</a> or <a href="format_occurrence.html">Occurrence</a>) with any applicable extension tables. Any biotic or abiotic measurements will be moved into the <a href="format_emof.html">extendedMeasurementOrFact table</a>. Before proceeding to the <a href="data_publication.html">publishing</a> stage, there are a number of <a href="dataquality.html">quality control</a> steps to complete.</p>
<p>Once your data has been published, you and others can <a href="access.html">access</a> datasets through various avenues and it becomes part of OBIS’ global database!</p>
<p>This may seem like a daunting process at first glance, but this manual will walk you through each step, and the OBIS community is full of <a href="gethelp.html">helpful resources</a>. Throughout the manual you will find tutorials and tools to guide you from start to finish through the OBIS data life cycle.</p>
<p><br clear="left"></p>
<section id="who-is-responsible-for-each-phase" class="level4" data-number="2.3.0.1">
<h4 data-number="2.3.0.1" class="anchored" data-anchor-id="who-is-responsible-for-each-phase"><span class="header-section-number">2.3.0.1</span> Who is responsible for each phase?</h4>
<p>Phases 1 through 3 are the responsibilities of the data provider, while Phases 3 and 4 are shared between the data provider and the node manager. Data users are involved in Phases 5 and 6.</p>
<p>The OBIS Secretariat is responsible for data processing and harvesting published resources.</p>
</section>
</section>
<section id="biodiversity-data-standards" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="biodiversity-data-standards"><span class="header-section-number">2.4</span> Biodiversity data standards</h2>
<p>From the very beginning, OBIS has championed the use of international standards for biogeographic data. Without agreement on the application of standards and protocols, OBIS would not have been able to build a large central database. OBIS uses the following standards:</p>
<ul>
<li><a href="darwin_core.html">Darwin Core</a></li>
<li><a href="eml.html">Ecological Metadata Language</a></li>
<li><a href="data_format.html">Darwin Core Archive and dataset structure</a></li>
</ul>
<p>The following pages of this manual review each of these in turn. We show you how to apply these standards to format your data in the <a href="formatting.html">Data Formatting</a> section.</p>
<p>We also provide some <a href="examples.html">dataset examples</a> for your reference.</p>
<section id="darwin-core" class="level3" data-number="2.4.1">
<h3 data-number="2.4.1" class="anchored" data-anchor-id="darwin-core"><span class="header-section-number">2.4.1</span> Darwin Core</h3>
<p><strong>Contents</strong></p>
<ul>
<li><a href="#introduction-to-darwin-core">Introduction to Darwin Core</a></li>
<li><a href="#darwin-core-terms">Darwin Core terms</a><br>
</li>
<li><a href="#darwin-core-guidelines">Darwin Core guidelines</a>
<ul>
<li><a href="#taxonomy-and-identification">Taxonomy and identification</a></li>
<li><a href="#occurrence">Occurrence</a><br>
</li>
<li><a href="#record-level-terms">Record level terms</a><br>
</li>
<li><a href="#location">Location</a><br>
</li>
<li><a href="#event">Event</a></li>
<li><a href="#time">Time</a><br>
</li>
<li><a href="#sampling">Sampling</a></li>
</ul></li>
</ul>
<section id="introduction-to-darwin-core" class="level4" data-number="2.4.1.1">
<h4 data-number="2.4.1.1" class="anchored" data-anchor-id="introduction-to-darwin-core"><span class="header-section-number">2.4.1.1</span> Introduction to Darwin Core</h4>
<p><a href="https://dwc.tdwg.org/">Darwin Core</a> is a body of standards (i.e., identifiers, labels, definitions) that facilitate sharing biodiversity informatics. It provides stable <a href="https://dwc.tdwg.org/terms/">terms</a> and vocabularies related to biological objects/data and their collection. Darwin Core is maintained by <a href="http://tdwg.org/">TDWG (Biodiversity Information Standards, formerly The International Working Group on Taxonomic Databases)</a>. Stable terms and vocabularies are important for ensuring the datasets in OBIS have consistently interpretable fields. By following Darwin Core standards, both data providers and users can be certain of the definition and quality of data.</p>
<section id="history-of-darwin-core-and-obis" class="level5" data-number="2.4.1.1.1">
<h5 data-number="2.4.1.1.1" class="anchored" data-anchor-id="history-of-darwin-core-and-obis"><span class="header-section-number">2.4.1.1.1</span> History of Darwin Core and OBIS</h5>
<p>The old <a href="http://old.iobis.org/node/304">OBIS schema</a> was an <a href="http://old.iobis.org/obis/obis.xsd">OBIS extension</a> to Darwin Core 1.2., which was based on <a href="http://rs.tdwg.org/dwc/terms/simple/">Simple Darwin Core</a>, a subset of Darwin Core which does not allow any structure beyond rows and columns. This old schema added some terms which were important for OBIS, but were not supported by Darwin Core at the time (e.g., start and end date and start and end latitude and longitude, depth range, lifestage, and terms for abundance, biomass and sample size).</p>
<p>In 2009, the Executive Committee of TDWG announced their ratification of an updated version of Darwin Core as a <a href="https://www.tdwg.org/community/dwc/#history-and-context">TDWG Standard</a>. Ratified Darwin Core unifies specializations and innovations emerging from diverse communities, and provides guidelines for ongoing enhancement. The <a href="https://dwc.tdwg.org/terms/">Darwin Core Quick Reference Guide</a> links to TDWG’s term definitions and related practices for Ratified Darwin Core. We will discuss the relevance of terms in this guide further below.</p>
<p>In December 2013, the <a href="https://obis.org/about/">3rd session of the IODE Steering Group for OBIS</a> agreed to transition OBIS globally to the TDWG-Ratified version of Darwin Core, and the mapping of the (old) OBIS specific terms to Darwin Core can be found <a href="http://rs.tdwg.org/dwc/terms/history/versions/index.htm#dwcobis">here</a>.</p>
<p><a class="anchor" name="terms"></a></p>
</section>
</section>
<section id="darwin-core-dwc-terms" class="level4" data-number="2.4.1.2">
<h4 data-number="2.4.1.2" class="anchored" data-anchor-id="darwin-core-dwc-terms"><span class="header-section-number">2.4.1.2</span> Darwin Core (DwC) terms</h4>
<p>DwC terms correspond to the column names of your dataset and can be grouped according to class type for convenience, e.g., Taxa, Occurrence, Record, Location, etc. It is important to use DwC field names because only columns using Darwin Core terms as headers will be recognized.</p>
<p>A list of all possible Darwin Core terms can be found on <a href="https://dwc.tdwg.org/terms/">TDWG</a>. However, OBIS does not parse all terms (note this doesn’t mean you cannot include them, they just will not be parsed when you publish to OBIS). Below is an overview of the most relevant Darwin Core terms to consider when contributing to OBIS, with guidelines regarding their use. We have also compiled a convenient <a href="checklist.html">checklist</a> of OBIS-accepted terms, their DwC class type, and which OBIS file (Event Core, Occurrence, eMoF, etc.) it is likely to be found in.</p>
<p>Note that OBIS currently has seven required and one strongly recommended DwC term: <code>occurrenceID</code>, <code>eventDate</code>, <code>decimalLongitude</code>, <code>decimalLatitude</code>, <code>scientificName</code>, <code>occurrenceStatus</code>, <code>basisOfRecord</code>, <code>scientificNameID</code> (strongly recommended).</p>
<p>The following DwC terms are related to the Class <em>Taxon</em>:</p>
<ul>
<li>scientificName</li>
<li>scientificNameID</li>
<li>scientificNameAuthorship</li>
<li>kingdom</li>
<li>taxonRank</li>
<li>taxonRemarks</li>
</ul>
<p>The following DwC terms are related to the Class <em>Identification</em>:</p>
<ul>
<li>identifiedBy</li>
<li>dateIdentified</li>
<li>identificationReferences</li>
<li>identificationRemarks</li>
<li>identificationQualifier</li>
<li>typeStatus</li>
</ul>
<p>The following DwC terms are related to the Class <em>Occurrence</em>:</p>
<ul>
<li>occurrenceID</li>
<li>occurrenceStatus</li>
<li>recordedBy</li>
<li>individualCount (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>organismQuantity (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>organismQuantityType (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>sex (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>lifeStage (OBIS recommends to add measurements to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>behavior</li>
<li>associatedTaxa</li>
<li>occurrenceRemarks</li>
<li>associatedMedia</li>
<li>associatedReferences</li>
<li>associatedSequences</li>
<li>catalogNumber</li>
<li>preparations</li>
</ul>
<p>The following DwC terms are related to the Class <em>Record level</em>:</p>
<ul>
<li>basisOfRecord</li>
<li>institutionCode</li>
<li>collectionCode</li>
<li>collectionID</li>
<li>bibliographicCitation</li>
<li>modified</li>
<li>dataGeneralizations</li>
</ul>
<p>The following DwC terms are related to the Class <em>Location</em>:</p>
<ul>
<li>decimalLatitude</li>
<li>decimalLongitude</li>
<li>coordinateUncertaintyInMeters</li>
<li>geodeticDatum</li>
<li>footprintWKT</li>
<li>minimumDepthInMeters</li>
<li>maximumDepthInMeters</li>
<li>minimumDistanceAboveSurfaceInMeters</li>
<li>maximumDistanceAboveSurfaceInMeters</li>
<li>locality</li>
<li>waterBody</li>
<li>islandGroup</li>
<li>island</li>
<li>country</li>
<li>locationAccordingTo</li>
<li>locationRemarks</li>
<li>locationID</li>
</ul>
<p>The following DwC terms are related to the Class <em>Event</em>:</p>
<ul>
<li>parentEventID</li>
<li>eventID</li>
<li>eventDate</li>
<li>type</li>
<li>habitat</li>
<li>samplingProtocol (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>sampleSizeValue (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>SampleSizeUnit (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
<li>samplingEffort (OBIS recommends to add sampling facts to <a href="data_format.html#extendedmeasurementorfact-extension-emof">eMoF</a>)</li>
</ul>
<p>The following DwC terms are related to the Class <em>MaterialSample</em>:</p>
<ul>
<li>materialSampleID</li>
</ul>
</section>
<section id="darwin-core-guidelines" class="level4" data-number="2.4.1.3">
<h4 data-number="2.4.1.3" class="anchored" data-anchor-id="darwin-core-guidelines"><span class="header-section-number">2.4.1.3</span> Darwin Core guidelines</h4>
<section id="taxonomy-and-identification" class="level5" data-number="2.4.1.3.1">
<h5 data-number="2.4.1.3.1" class="anchored" data-anchor-id="taxonomy-and-identification"><span class="header-section-number">2.4.1.3.1</span> Taxonomy and identification</h5>
<p><code>scientificName</code> (required term) should always contain the originally recorded scientific name, even if the name is currently a synonym. This is necessary to be able to track back records to the original dataset. The name should be at the lowest possible taxonomic rank, preferably at species level or lower, but higher ranks, such as genus, family, order, class etc. are also acceptable. We recommend to not include authorship in <code>scientificName</code>, and only use <code>scientificNameAuthorship</code> for that purpose. The <code>scientificName</code> term should only contain the name and not identification qualifications (such as ?, confer or affinity), which should instead be supplied in the <code>IdentificationQualifier</code> term, see examples below. <code>taxonRemarks</code> can capture comments or notes about the taxon or name.</p>
<p>A <a href="http://www.marinespecies.org/">WoRMS</a> LSID should be added in <code>scientificNameID</code> (strongly recommended term), OBIS will use this identifier to pull the taxonomic information from the World Register of Marine Species (WoRMS) into OBIS and attach it to your dataset. This information includes:</p>
<ul>
<li>Taxonomic classification (kingdom through species)</li>
<li>The accepted name in case of invalid names or synonyms</li>
<li>AphiaID</li>
<li>IUCN red list category</li>
</ul>
<p>LSIDs are persistent, location-independent, resource identifiers for uniquely naming biologically significant resources. More information on LSIDs can be found at <a href="http://www.lsid.info/">www.lsid.info</a>. For example, the WoRMS LSID for <em>Solea solea</em> is: urn:lsid:marinespecies.org:taxname:127160, and can be found at the bottom of each WoRMS taxon page, e.g.&nbsp;<a href="http://marinespecies.org/aphia.php?p=taxdetails&amp;id=127160"><em>Solea solea</em></a>.</p>
<p><code>kingdom</code> and <code>taxonRank</code> can help us in identifying the provided <code>scientificName</code> in case the name is not available in WoRMS. <code>kingdom</code> in particular can help us find alternative genus-species combinations and avoids linking the name to homonyms. Please contact the WoRMS data management team (info@marinespecies.org) in case the scientificName is missing in WoRMS. <code>kingdom</code> and <code>taxonRank</code> are not necessary when a correct <code>scientificNameID</code> is provided.</p>
<p>OBIS recommends providing information about how an identification was made, for example by which ID key, species guide or expert; and by which method (e.g morphology vs.&nbsp;genomics), etc. The person’s name who made the taxonomic identification can go in <code>identifiedBy</code> and <em>when</em> in <code>dateIdentified</code>. Use the ISO 8601:2004(E) standard for date and time, for instructions see <a href="#time">Time</a>. A list of references, such as field guides used for the identification can be listed in <code>identificationReferences</code>. Any other information, such as identification methods, can be added to <code>identificationRemarks</code>.</p>
<p>Examples:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption></caption>
<colgroup>
<col style="width: 43%">
<col style="width: 23%">
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 12%">
</colgroup>
<thead>
<tr class="header">
<th>scientificNameID</th>
<th>scientificName</th>
<th>kingdom</th>
<th>phylum</th>
<th>class</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>urn:lsid:marinespecies.org:taxname:142004</td>
<td>Yoldiella nana</td>
<td>Animalia</td>
<td>Mollusca</td>
<td>Bivalvia</td>
</tr>
<tr class="even">
<td>urn:lsid:marinespecies.org:taxname:140584</td>
<td>Ennucula tenuis</td>
<td>Animalia</td>
<td>Mollusca</td>
<td>Bivalvia</td>
</tr>
<tr class="odd">
<td>urn:lsid:marinespecies.org:taxname:131573</td>
<td>Terebellides stroemii</td>
<td>Animalia</td>
<td>Annelida</td>
<td>Polychaeta</td>
</tr>
</tbody>
</table>
</div>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption><em>Data from <a href="http://ipt.vliz.be/eurobis/resource?r=largenet_k2">Benthic fauna around Franz Josef Land</a>.</em></caption>
<thead>
<tr class="header">
<th>order</th>
<th>family</th>
<th>genus</th>
<th>specificEpithet</th>
<th>scientificNameAuthorship</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Nuculanoida</td>
<td>Yoldiidae</td>
<td>Yoldiella</td>
<td>nana</td>
<td>(Sars M., 1865)</td>
</tr>
<tr class="even">
<td>Nuculoida</td>
<td>Nuculidae</td>
<td>Ennucula</td>
<td>tenuis</td>
<td>(Montagu, 1808)</td>
</tr>
<tr class="odd">
<td>Terebellida</td>
<td>Trichobranchidae</td>
<td>Terebellides</td>
<td>stroemii</td>
<td>Sars, 1835</td>
</tr>
</tbody>
</table>
</div>
<p>If the record represents a nomenclatural type specimen, the term <code>typeStatus</code> can be used, e.g.&nbsp;for holotype, syntype, etc.</p>
<p><strong>In case of low confidence identifications</strong>, and the scientific name contains qualifiers such as <em>cf.</em>, <em>?</em> or <em>aff.</em>, then this name should go in <code>identificationQualifier</code>, and <code>scientificName</code> should contain the name of the lowest possible taxon rank that refers to the most accurate identification. E.g. if the specimen was accurately identified down to genus level, but not species level, then the scientificName should contain the name of the genus, the scientificNameID should contain the LSID the genus and the <code>identificationQualifier</code> should contain the low confidence species name combined with <em>?</em> or other qualifiers. The table below shows a few examples:</p>
<p>The use and definitions for additional ON signs (<code>identificationQualifier</code>) can be found in <a href="https://doi.org/10.1111/2041-210X.12594">Open Nomenclature in the biodiversity era</a>, which provides examples for using the main Open Nomenclature qualifiers associated with <em>physical specimens</em>. The publication <a href="https://www.frontiersin.org/articles/10.3389/fmars.2021.620702/full">Recommendations for the Standardisation of Open Taxonomic Nomenclature for Image-Based Identiﬁcations</a> provides examples and definitions for <code>identificationQualifiers</code> for <em>non-physical specimens (image-based)</em>.</p>
<p>Examples:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption></caption>
<colgroup>
<col style="width: 11%">
<col style="width: 22%">
<col style="width: 24%">
<col style="width: 6%">
<col style="width: 14%">
<col style="width: 20%">
</colgroup>
<thead>
<tr class="header">
<th>scientificName</th>
<th>scientificNameAuthorship</th>
<th>scientificNameID</th>
<th>taxonRank</th>
<th>identificationQualifier</th>
<th>taxonConceptID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Pelagia</td>
<td>Péron &amp; Lesueur, 1810</td>
<td>urn:lsid:marinespecies.org:taxname:135262</td>
<td>genus</td>
<td>gen. nov.</td>
<td>Pelagia gen. nov.</td>
</tr>
<tr class="even">
<td>Pelagia benovici</td>
<td>Piraino, Aglieri, Scorrano &amp; Boero, 2014</td>
<td>urn:lsid:marinespecies.org:taxname:851656</td>
<td>species</td>
<td>sp. nov</td>
<td>Pelagia benovici sp. nov</td>
</tr>
<tr class="odd">
<td>Gadus</td>
<td>Linnaeus, 1758</td>
<td>urn:lsid:marinespecies.org:taxname:125732</td>
<td>genus</td>
<td>cf.&nbsp;morhua</td>
<td>Gadus cf.&nbsp;morhua</td>
</tr>
<tr class="even">
<td>Polycera</td>
<td>Cuvier, 1816</td>
<td>urn:lsid:marinespecies.org:taxname:138369</td>
<td>genus</td>
<td>cf.&nbsp;hedgpethi</td>
<td>Polycera cf.&nbsp;hedgpethi</td>
</tr>
<tr class="odd">
<td>Tubifex</td>
<td>Lamarck, 1816</td>
<td>urn:lsid:marinespecies.org:taxname:137392</td>
<td>genus</td>
<td>?</td>
<td>Tubifex tubifex(Müller, 1774)?</td>
</tr>
<tr class="even">
<td>Tubifex</td>
<td>Lamarck, 1816</td>
<td>urn:lsid:marinespecies.org:taxname:137392</td>
<td>genus</td>
<td>sp. inc.</td>
<td>Tubifex tubifex(Müller, 1774)sp. inc.</td>
</tr>
<tr class="odd">
<td>Brisinga</td>
<td>Asbjørnsen, 1856</td>
<td>urn:lsid:marinespecies.org:taxname:123210</td>
<td>genus</td>
<td>gen. inc.</td>
<td>Brisinga gen. inc.</td>
</tr>
<tr class="even">
<td>Uroptychus compressus</td>
<td>Baba &amp; Wicksten, 2019</td>
<td>urn:lsid:marinespecies.org:taxname:1332465</td>
<td>genus</td>
<td>sp. inc.</td>
<td>Uroptychus compressus sp. inc.</td>
</tr>
<tr class="odd">
<td>Eurythenes</td>
<td>S. I. Smith in Scudder, 1882</td>
<td>urn:lsid:marinespecies.org:taxname:101607</td>
<td>genus</td>
<td>sp. DISCOLL.PAP.JC165.674</td>
<td>Eurythenes sp.DISCOLL.PAP.JC165.674</td>
</tr>
<tr class="even">
<td>Paroriza</td>
<td>Hérouard, 1902</td>
<td>urn:lsid:marinespecies.org:taxname:123467</td>
<td>genus</td>
<td>sp.[unique123]aff.pallens</td>
<td>Paroriza sp.[unique123]aff. pallens</td>
</tr>
<tr class="odd">
<td>Aristeidae</td>
<td>Wood-Mason in Wood-Mason &amp; Alcock, 1891</td>
<td>urn:lsid:marinespecies.org:taxname:106725</td>
<td>family</td>
<td>stet.</td>
<td>Aristeidae stet.</td>
</tr>
<tr class="even">
<td>Nematocarcinus</td>
<td>Milne-Edwards, 1881</td>
<td>urn:lsid:marinespecies.org:taxname:107015</td>
<td>genus</td>
<td>sp.indet.</td>
<td>Nematocarcinus sp.indet.</td>
</tr>
<tr class="odd">
<td>Brisinga</td>
<td>Asbjørnsen, 1856</td>
<td>urn:lsid:marinespecies.org:taxname:123210</td>
<td>genus</td>
<td>gen.inc.</td>
<td>Brisinga gen.inc.</td>
</tr>
<tr class="even">
<td>Brisinga costata</td>
<td>Verrill, 1884</td>
<td>urn:lsid:marinespecies.org:taxname:17825</td>
<td>species</td>
<td>sp.inc.</td>
<td>Brisinga costata sp.inc.</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="occurrence" class="level5" data-number="2.4.1.3.2">
<h5 data-number="2.4.1.3.2" class="anchored" data-anchor-id="occurrence"><span class="header-section-number">2.4.1.3.2</span> Occurrence</h5>
<p><code>occurrenceID</code> (required term) is an identifier for the occurrence record and should be persistent and globally unique. If the dataset does not yet contain (globally unique) occurrenceIDs, then they should be created. Guideline for ID creation can be found <a href="identifiers.html">here</a></p>
<p><code>occurrenceStatus</code> (required term) is a statement about the presence or absence of a taxon at a location. It is an important term, because it allows us to distinguish between presence and absence records. It is a required term and should be filled in with either <code>present</code> or <code>absent</code>.</p>
<p>A few terms related to quantity: <code>organismQuantity</code> and <code>organismQuantityType</code>, have been added to the TDWG ratified Darwin Core. This is a lot more versatile than the older <code>individualCount</code> field. However, OBIS recommends to use the <a href="data_format.html#extendedmeasurementorfact-extension-emof">Extended MeasurementorFact extension</a> for quantitative measurements because of the standardization of terms and the fact that you can link these measurements to sampling events and factual sampling information.</p>
<p>Please take note that OBIS recommends all quantitative measurements and sampling facts to be placed in the <code>ExtendedMeasurementOrFact</code> extension and not in the Darwin Core files.</p>
<p>In the case specimens were collected and stored (e.g.&nbsp;museum collections), the <code>catalogNumber</code> and <code>preparations</code> terms can be used to provide the identifier for the record in the collection and to document the preparation and preservation methods. The term <code>typeStatus</code> see above (under identification) can be used in this context too.</p>
<p>Both <code>associatedMedia</code>, <code>associatedReferences</code> and <code>associatedSequences</code> are global unique identifiers or URIs pointing to respectively associated media (e.g.&nbsp;online image or video), associated literature (e.g.&nbsp;DOIs) or genetic sequence information (e.g.&nbsp;GenBANK ID).</p>
<p><code>associatedTaxa</code> include a list (concatenated and separated) of identifiers or names of taxa and their associations with the Occurrence, e.g.&nbsp;the species occurrence was associated to the presence of kelp such as <em>Laminaria digitata</em>.</p>
<p>The recommended vocabulary for <code>sex</code> see <a href="http://vocab.nerc.ac.uk/collection/S10/current/">BODC vocab : S10</a>, for <code>lifeStage</code> see <a href="http://vocab.nerc.ac.uk/collection/S11/current/">BODC vocab: S11</a>, <code>behavior</code> (no vocab available), and <code>occurrenceRemarks</code> can hold any comments or notes about the Occurrence.</p>
<p><code>recordedBy</code> can hold a list (concatenated and separated) of names of people, groups, or organizations responsible for recording the original Occurrence. The primary collector or observer, especially one who applies a personal identifier (recordNumber), should be listed first.</p>
<p>Example:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption><em>Data from <a href="http://ipt.vliz.be/eurobis/resource?r=summary_of_benthic_studies_in_the_sluice_dock_ostend_during_1976-1981">A summary of benthic studies in the sluice dock of Ostend during 1976-1981</a>.</em></caption>
<thead>
<tr class="header">
<th>collectionCode</th>
<th>occurrenceID</th>
<th>catalogNumber</th>
<th>occurrenceStatus</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SluiceDock_benthic_1976/1981</td>
<td>SluiceDock_benthic_1976_1</td>
<td>SluiceDock_benthic_1976_1</td>
<td>present</td>
</tr>
<tr class="even">
<td>SluiceDock_benthic_1976/1981</td>
<td>SluiceDock_benthic_1976_2</td>
<td>SluiceDock_benthic_1976_2</td>
<td>present</td>
</tr>
<tr class="odd">
<td>SluiceDock_benthic_1976/1981</td>
<td>SluiceDock_benthic_1979-07/1980-06_1</td>
<td>SluiceDock_benthic_1979-07/1980-06_1</td>
<td>present</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="record-level-terms" class="level5" data-number="2.4.1.3.3">
<h5 data-number="2.4.1.3.3" class="anchored" data-anchor-id="record-level-terms"><span class="header-section-number">2.4.1.3.3</span> Record level terms</h5>
<p><code>basisOfRecord</code> (required term) specifies the nature of the record, i.e.&nbsp;whether the occurrence record is based on a stored specimen or an observation. In case the specimen is collected and stored in a collection (e.g.&nbsp;at a museum, university, research institute), the options are:</p>
<ul>
<li><code>PreservedSpecimen</code> e.g.&nbsp;preserved in ethanol, tissue etc.</li>
<li><code>FossilSpecimen</code> a fossil, which allows OBIS to make the distinction between the date of collection and the time period the specimen was assumed alive</li>
<li><code>LivingSpecimen</code> an intentionally kept/cultivated living specimen e.g.&nbsp;in an aquarium or culture collection.</li>
</ul>
<p>In case no specimen is deposited, the basis of record is either <code>HumanObservation</code> (e.g bird sighting, benthic sample but specimens were discarded after counting), or <code>MachineObservation</code> (e.g.&nbsp;for occurrences based on automated sensors such as image recognition, etc). For records pertaining to genetic samples, basisOfRecord can be <code>MaterialSample</code> (e.g.&nbsp;in the DNA-derived data extension).</p>
<p>When the basisOfRecord is either a <em>preservedSpecimen</em>, <em>LivingSpecimen</em> or <em>FossilSpecimen</em> please also add the <code>institutionCode</code>, <code>collectionCode</code> and <code>catalogNumber</code>, which will enable people to visit the collection and re-examine the material. Sometimes, for example in case of living specimens, a dataset can contain records pointing to the origin, the in-situ sampling position as well as a record referring to the ex-situ collection. In this case please add the event type information in <code>eventRemarks</code> (see <a href="darwin_core.html#event">OBIS manual: event</a>).</p>
<p><code>institutionCode</code> identifies the custodian institute (often by acronym), <code>collectionCode</code> identifies the collection or dataset within that institute. Collections cannot belong to multiple institutes, so all records within a collection should have the same <code>institutionCode</code>. The <code>collectionID</code> is an identifier for the record within the dataset or collection.</p>
<p><code>bibliographicCitation</code> allows for providing different citations on record level, while a single citation for the entire dataset can and should be provided in the metadata (see <a href="eml.html">EML</a>). The citation at record level can have the format of a chapter in a book, where the book is the dataset citation. The record citation will have preference over the dataset citation. We do not, however, recommend to create different citations for every record, as this will explode the number of citations and will hamper the re-use of data.</p>
<p><code>modified</code> is the most recent date-time on which the resource was changed. It is required to use the ISO 8601:2004(E) standard, for instructions see <a href="#time">Time</a>.</p>
<p><code>dataGeneralizations</code> refers to actions taken to make the shared data less specific or complete than in its original form. Suggests that alternative data of higher quality may be available on request. This can be the case for occurrences of vulnerable or endangered species and there positions are converted to the center of grid cells.</p>
<p><a class="anchor" name="location"></a></p>
</section>
<section id="location" class="level5" data-number="2.4.1.3.4">
<h5 data-number="2.4.1.3.4" class="anchored" data-anchor-id="location"><span class="header-section-number">2.4.1.3.4</span> Location</h5>
<p><code>decimalLatitude</code> and <code>decimalLongitude</code> (required terms) are the geographic latitude and longitude (in decimal degrees), using the spatial reference system given in <code>geodeticDatum</code> of the geographic center of a Location. The number of decimals should be appropriate for the level of uncertainty in <code>coordinateUncertaintyInMeters</code> (at least within an order of magnitude). <code>coordinateUncertaintyInMeters</code> is the radius of the smallest circle around the given position containing the whole location. Regarding <code>decimalLatitude</code>, positive values are north of the Equator, negative values are south of it. All values lie between -90 and 90, inclusive. Regarding <code>decimalLongitude</code>, positive values are east of the Greenwich Meridian, negative values are west of it. All values lie between -180 and 180, inclusive.</p>
<p>In OBIS, the spatial reference system to be documented in <code>geodeticDatum</code> is <a href="https://epsg.io/4326">EPSG:4326</a>. Coordinates in degrees/minutes/seconds can be converted to decimal degrees using our <a href="http://iobis.github.io/coordinates/">coordinates tool</a>. We also provide a <a href="https://obis.org/maptool">tool</a> to check coordinates or to determine coordinates for a location (point, transect or polygon) on a map. This tool also allows geocoding location names using <a href="http://www.marineregions.org/">marineregions.org</a>.</p>
<p>The name of the place or location can be provided in <code>locality</code>, and if possible linked by a <code>locationID</code> using a persistent ID from a gazetter, such as the MRGID from <a href="http://www.marineregions.org/gazetteer.php?p=search">MarineRegions</a>. If the species occurrence only contains the name of the <code>locality</code>, but not the exact coordinates, we recommend using a geocoding service to obtain the coordinates. <a href="http://www.marineregions.org/">Marine Regions</a> has a <a href="http://www.marineregions.org/gazetteer.php?p=search">search interface</a> for geographic names, and provides coordinates and often precision in meters, which can go into <code>coordinateUncertaintyInMeters</code>. Another option is to use the <a href="http://www.getty.edu/research/tools/vocabularies/tgn/">Getty Thesaurus of Geographic Names</a> or <a href="http://maps.google.com">Google Maps</a>: after looking up a location, the decimal coordinates can be found in the page URL. Additional information about the locality can also be stored in DwC terms such as <code>waterBody</code>, <code>islandGroup</code>, <code>island</code> and <code>country</code>. <code>locationAccordingTo</code> should provide the name of the gazetteer that is used to obtain the coordinates for the locality.</p>
<p><code>locationID</code> is an identifier for the set of location information (e.g.&nbsp;station ID, or MRGID from <a href="http://www.marineregion.org">marineregions</a>), for example the <a href="http://www.marineregions.org/gazetteer.php?p=details&amp;id=3956">Balearic Plain</a> has MRGID: <a href="http://marineregions.org/mrgid/3956">http://marineregions.org/mrgid/3956</a>.</p>
<p>A <a href="https://en.wikipedia.org/wiki/Well-known_text">Well-Known Text</a> (WKT) representation of the shape of the location can be provided in <code>footprintWKT</code>. This is particularly useful for tracks, transects, tows, trawls, habitat extent or when an exact location is not known. WKT strings can be created using our <a href="https://obis.org/maptool">WKT tool</a>. This tool also calculates a midpoint and a radius, which can then be added to <code>decimalLongitude</code>, <code>decimalLatitude</code>, and <code>coordinateUncertaintyInMeters</code> respectively. There is also an <a href="https://github.com/iobis/obistools#calculate-centroid-and-radius-for-wkt-geometries">R tool</a> to calculate the centroid and radius for WKT polygons. <a href="https://wktmap.com">wktmap.com</a> can be used to visualize and share WKT strings.</p>
<!--<img src="images/wkttool2.png" class="img-responsive"/>-->
<p><img src="images/wkttool2.png" class="img-fluid"></p>
<p>Some examples of WKT strings:</p>
<pre class="text"><code>LINESTRING (30 10, 10 30, 40 40)
POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))
MULTILINESTRING ((10 10, 20 20, 10 40),(40 40, 30 30, 40 20, 30 10))
MULTIPOLYGON (((30 20, 45 40, 10 40, 30 20)),((15 5, 40 10, 10 20, 5 10, 15 5)))</code></pre>
<p>Example:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption><em>Data from <a href="http://ipt.vliz.be/eurobis/resource?r=ionian_2008_2018">Adriatic and Ionian Sea mega-fauna monitoring employing ferry as platform of observation along the Ancona-Igoumenitsa-Patras lane, from December 2014 to December 2018</a>.</em></caption>
<thead>
<tr class="header">
<th>decimalLatitude</th>
<th>decimalLongitude</th>
<th>geodeticDatum</th>
<th>coordinateUncertaintyInMeters</th>
<th>footprintWKT</th>
<th>footprintSRS</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>38.698</td>
<td>20.95</td>
<td>EPSG:4326</td>
<td>75033.17</td>
<td>LINESTRING (20.31 39.15, 21.58 38.24)</td>
<td>EPSG:4326</td>
</tr>
<tr class="even">
<td>42.72</td>
<td>15.228</td>
<td>EPSG:4326</td>
<td>154338.87</td>
<td>LINESTRING (16.64 41.80, 13.82 43.64)</td>
<td>EPSG:4326</td>
</tr>
<tr class="odd">
<td>39.292</td>
<td>20.364</td>
<td>EPSG:4326</td>
<td>162083.27</td>
<td>LINESTRING (19.05 40.34, 21.68 38.25)</td>
<td>EPSG:4326</td>
</tr>
</tbody>
</table>
</div>
<p>Keep in mind while filling in <a href="http://rs.tdwg.org/dwc/terms/maximumDepthInMeters"><code>minimumDepthInMeters</code></a> and <a href="http://rs.tdwg.org/dwc/terms/minimumDepthInMeters"><code>maximumDepthInMeters</code></a> that this should be the depth at which the <strong>sample was taken</strong> and not the water column depth at that location. When fillling in any depth fields (<code>minimumDepthInMeters</code>, <code>maximumDepthInMeters</code>, <a href="http://rs.tdwg.org/dwc/terms/minimumDistanceAboveSurfaceInMeters"><code>minimumDistanceAboveSurfaceInMeters</code></a>, and <a href="http://rs.tdwg.org/dwc/terms/maximumDistanceAboveSurfaceInMeters"><code>maximumDistanceAboveSurfaceInMeters</code></a>), you should also consider which information is needed to fully understand the data. In most cases (e.g.&nbsp;scenario 1 and 4 in the figure below), providing <code>minimumDepthInMeters</code> and <code>maximumDepthInMeters</code> is sufficient for observations of organisms at particular depths. However, in cases where an occurrence is above the sea surface, e.g.&nbsp;flying birds (scenario 2 and 5), you should populate <code>minimumDistanceAboveSurfaceInMeters</code>, <code>maximumDistanceAboveSurfaceInMeters</code>, and, where relevant, you should also include <a href="https://dwc.tdwg.org/list/#dwc_minimumElevationInMeters"><code>minimumElevationInMeters</code></a> and <a href="https://dwc.tdwg.org/list/#dwc_maximumElevationInMeters"><code>maximumElevationInMeters</code></a>.</p>
<p>The <code>minimumDistanceAboveSurfaceInMeters</code> and <code>maximumDistanceAboveSurfaceInMeters</code> is the distance, in meters, above or below a reference surface or reference point. The reference surface is determined by the depth or elevation. If the depth and elevation are 0, then the reference surface is the sea surface. If a depth is given, the reference surface is the location of the depth. This can be especially useful for sediment cores taken from the sea bottom (scenario 3 in figure below). If no depth is given, then the elevation is the reference surface (scenario 5).</p>
<p><img src="images/Depth-figure-updated.png" class="img-fluid"></p>
<p>Depth scenario examples:</p>
<div class="table-responsive">
<table class="table-striped table-hover table">
<caption></caption>
<colgroup>
<col style="width: 6%">
<col style="width: 13%">
<col style="width: 13%">
<col style="width: 22%">
<col style="width: 22%">
<col style="width: 10%">
<col style="width: 10%">
</colgroup>
<thead>
<tr class="header">
<th>Scenario</th>
<th>minimumDepthInMeters</th>
<th>maximumDepthInMeters</th>
<th>minimumDistanceAboveSurfaceInMeters</th>
<th>maximumDistanceAboveSurfaceInMeters</th>
<th>minimumElevationInMeters</th>
<th>maximumElevationInMeters</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>40, 90</td>
<td>50, 100</td>
<td>-</td>
<td>-</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="even">
<td>2</td>
<td>0</td>
<td>0</td>
<td>10</td>
<td>15</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="odd">
<td>3</td>
<td>100</td>
<td>100</td>
<td>0</td>
<td>-1.5</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="even">
<td>4</td>
<td>20</td>
<td>22</td>
<td>-</td>
<td>-</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="odd">
<td>5</td>
<td>0</td>
<td>0</td>
<td>10</td>
<td>15</td>
<td>10</td>
<td>10</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="event" class="level5" data-number="2.4.1.3.5">
<h5 data-number="2.4.1.3.5" class="anchored" data-anchor-id="event"><span class="header-section-number">2.4.1.3.5</span> Event</h5>
<p><code>eventID</code> is an identifier for the sampling or observation event. <code>parentEventID</code> is an identifier for a parent event, which is composed of one or more sub-sampling (child) events (eventIDs). See <a href="identifiers.html#eventid">identifiers</a> for details on how these terms can be constructed.</p>
<p><code>habitat</code> is a category or description of the habitat in which the Event occurred (e.g.&nbsp;benthos, seamount, hydrothermal vent, seagrass, rocky shore, intertidal, ship wreck etc.)</p>
</section>
<section id="time" class="level5" data-number="2.4.1.3.6">
<h5 data-number="2.4.1.3.6" class="anchored" data-anchor-id="time"><span class="header-section-number">2.4.1.3.6</span> Time</h5>
<p>The date and time at which an occurrence was recorded goes in <code>eventDate</code>. This term uses the <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601 standard</a> and OBIS recommends using the extended ISO 8601 format with hyphens.</p>
<!--<img src="https://imgs.xkcd.com/comics/iso_8601.png" class="img-responsive"/>-->
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="https://imgs.xkcd.com/comics/iso_8601.png" class="img-fluid figure-img"></p>
</figure>
</div>
<p>More specific guidelines on formatting dates and times can be found in the <a href="common_formatissues.html#temporal-dates-and-times">Common Data formatting issues page</a></p>
</section>
<section id="sampling" class="level5" data-number="2.4.1.3.7">
<h5 data-number="2.4.1.3.7" class="anchored" data-anchor-id="sampling"><span class="header-section-number">2.4.1.3.7</span> Sampling</h5>
<p>Information on <code>sampleSizeValue</code> and <code>sampleSizeUnit</code> is very important when an organism quantity is specified. However, with <a href="data_format.html">OBIS-ENV-DATA</a> it was felt that the extended MeasurementorFact (<a href="http://rs.gbif.org/extension/obis/extended_measurement_or_fact.xml">eMoF</a>) extension would be better suited than the DwC Event Core to store the sampled area and/or volume because in some cases sampleSize by itself may not be detailed enough to allow interpretation of the sample. For instance, in the case of a plankton tow, the volume of water that passed through the net is relevant. In case of Niskin bottles, the volume of sieved water is more relevant than the actual volume in the bottle. In these examples, as well as generally when recording sampling effort for all protocols, eMoF enables greater flexibility to define parameters, as well as the ability to describe the entire sample and treatment protocol through multiple parameters. eMoF also allows you to standardize your terms to a controlled vocabulary.</p>
<p>The next chapter deals with the metadata (description of the dataset) in <a href="eml.html">Ecological Metadata Language</a>.</p>
</section>
</section>
</section>
<section id="darwin-core-archive" class="level3" data-number="2.4.2">
<h3 data-number="2.4.2" class="anchored" data-anchor-id="darwin-core-archive"><span class="header-section-number">2.4.2</span> Darwin Core Archive</h3>
<p><strong>Contents</strong></p>
<ul>
<li><a href="#darwin-core-archive">Darwin Core Archive</a></li>
<li><a href="#obis-holds-more-than-just-species-occurrences-the-env-data-approach">OBIS holds more than just species occurrences: the ENV-DATA approach</a>
<ul>
<li><a href="#extendedmeasurementorfact-extension-emof">ExtendedMeasurementOrFact Extension (eMoF)</a></li>
<li><a href="#edna-dna-derived-data-extension">eDNA &amp; DNA derived data Extension</a></li>
<li><a href="#a-special-case-habitat-types">A special case: habitat types</a></li>
</ul></li>
<li><a href="#recommended-reading">Recommended reading</a></li>
</ul>
<section id="darwin-core-archive-1" class="level4" data-number="2.4.2.1">
<h4 data-number="2.4.2.1" class="anchored" data-anchor-id="darwin-core-archive-1"><span class="header-section-number">2.4.2.1</span> Darwin Core Archive</h4>
<p>Darwin Core Archive (DwC-A) is the standard for packaging and publishing biodiversity data using Darwin Core terms. It is the preferred format for publishing data in OBIS and GBIF. The format is described in the <a href="https://dwc.tdwg.org/text/">Darwin Core text guide</a>. A Darwin Core Archive contains a number of text files, including data tables formatted as CSV.</p>
<p>The conceptual data model of the Darwin Core Archive is a star schema with a single core table, for example containing occurrence records or event records, at the center of the star. Extension tables can optionally be associated with the core table. It is not possible to link extension tables to other extension tables (to form a so-called snowflake schema). There is a one-to-many relationship between the core and extension records, so each core record can have zero or more extension records linked to it, and each extension record must be linked to exactly one core record. Definitions for the core and extension tables can be found <a href="http://rs.gbif.org/">here</a>.</p>
<p>Besides data tables, a Darwin Core Archive also contains two XML files: one file which describes the archive and data file structure (<code>meta.xml</code>), and one file which contains the dataset’s metadata (<code>eml.xml</code>).</p>
<img src="images/dwca.png" class="img-responsive img-responsive-50">
<p class="caption-50">
Figure: structure of a Darwin Core Archive.
</p>
</section>
<section id="obis-holds-more-than-just-species-occurrences-the-env-data-approach" class="level4" data-number="2.4.2.2">
<h4 data-number="2.4.2.2" class="anchored" data-anchor-id="obis-holds-more-than-just-species-occurrences-the-env-data-approach"><span class="header-section-number">2.4.2.2</span> OBIS holds more than just species occurrences: the ENV-DATA approach</h4>
<p>Data collected as part of marine biological research often include measurements of habitat features (such as physical and chemical parameters of the environment), biotic and biometric measurements (such as body size, abundance, biomass), as wel as details regarding the nature of the sampling or observation methods, equipment, and sampling effort.</p>
<p>In the past, OBIS relied solely on the <a href="http://rs.gbif.org/core/dwc_occurrence_2015-07-02.xml">Occurrence Core</a>, and additional measurements were added in a structured format (e.g.&nbsp;JSON) in the Darwin Core term <code>dynamicProperties</code> inside the occurrence records. This approach had significant downsides: the format is difficult to construct and deconstruct, there is no standardization of terms, and attributes which are shared by multiple records (think sampling methodology) have to be repeated many times. The formatting problem can be addressed by moving measurements to a <a href="http://rs.gbif.org/extension/dwc/measurements_or_facts.xml">MeasurementOrFacts</a> extension table, but that doesn’t solve the redundancy and standardization problems.</p>
<p>With the release and adoption of a new core type <a href="http://rs.gbif.org/core/dwc_event_2015_05_29.xml">Event Core</a> it became possible to associate measurements with nested events (such as cruises, stations, and samples), but the restrictive star schema of Darwin Core archive prohibited associating measurements with the event records in the Event core as well as with the occurrence records in the Occurrence extension. For this reason an extended version of the existing MeasurementOrFact extension was created.</p>
<section id="extendedmeasurementorfact-extension-emof" class="level5" data-number="2.4.2.2.1">
<h5 data-number="2.4.2.2.1" class="anchored" data-anchor-id="extendedmeasurementorfact-extension-emof"><span class="header-section-number">2.4.2.2.1</span> ExtendedMeasurementOrFact Extension (eMoF)</h5>
<p>As part of the IODE pilot project <a href="https://www.iode.org/index.php?option=com_content&amp;view=article&amp;id=463&amp;Itemid=100200">Expanding OBIS with environmental data OBIS-ENV-DATA</a>, OBIS introduced a custom <a href="http://rs.gbif.org/extension/obis/extended_measurement_or_fact.xml">ExtendedMeasurementOrFact</a> or eMoF extension, which extends the existing <a href="http://rs.gbif.org/extension/dwc/measurements_or_facts.xml">MeasurementOrFact</a> extension with 4 new terms:</p>
<ul>
<li><code>occurrenceID</code></li>
<li><code>measurementTypeID</code></li>
<li><code>measurementValueID</code></li>
<li><code>measurementUnitID</code></li>
</ul>
<p>The <code>occurrenceID</code> term is used to circumvent the limitations of the star schema, and link measurement records in the ExtendedMeasurementOrFact extension to occurrence records in the Occurrence extension. Note that in order to comply with the Darwin Core Archive standard, these records still need to link to an event record in the Event core table as well. Thanks to this term we can now store a variety of measurements and facts linked to either events or occurrences:</p>
<ul>
<li>organism quantifications (e.g.&nbsp;counts, abundance, biomass, % live cover, etc.)</li>
<li>species biometrics (e.g.&nbsp;body length, weight, etc.)</li>
<li>facts documenting a specimen (e.g.&nbsp;living/dead, behaviour, invasiveness, etc.)</li>
<li>abiotic measurements (e.g.&nbsp;temperature, salinity, oxygen, sediment grain size, habitat features)</li>
<li>facts documenting the sampling activity (e.g.&nbsp;sampling device, sampled area, sampled volume, sieve mesh size).</li>
</ul>
<img src="images/EventCoreSchema.png" class="img-responsive img-responsive-50">
<p class="caption-50">
Figure: Overview of an OBIS-ENV-DATA format. Sampling parameters, abiotic measurements, and occurrences are linked to events using the eventID (full lines). Biotic measurements are linked to occurrences using the new occurrenceID field of the ExtendedMeasurementOrFact Extension (dashed lines).
</p>
</section>
<section id="edna-dna-derived-data-extension" class="level5" data-number="2.4.2.2.2">
<h5 data-number="2.4.2.2.2" class="anchored" data-anchor-id="edna-dna-derived-data-extension"><span class="header-section-number">2.4.2.2.2</span> eDNA &amp; DNA derived data Extension</h5>
<p>DNA derived data are increasingly being used to document taxon occurrences. To ensure these data are useful to the broadest possible community, GBIF published a guide entitled <a href="https://docs.gbif-uat.org/publishing-dna-derived-data/1.0/en/">Publishing DNA-derived data through biodiversity data platforms</a>. This guide is supported by the DNA derived data extension for Darwin Core, which incorporates MIxS terms into the Darwin Core standard. eDNA and DNA derived data is linked to occurrence data with the use of <code>occurrenceID</code> and/ or <code>eventID</code>. Refer to the <a href="dna_data.html">Examples: ENV-DATA and DNA derived data</a> for use case examples of eDNA and DNA derived data.</p>
</section>
<section id="a-special-case-habitat-types" class="level5" data-number="2.4.2.2.3">
<h5 data-number="2.4.2.2.3" class="anchored" data-anchor-id="a-special-case-habitat-types"><span class="header-section-number">2.4.2.2.3</span> A special case: habitat types</h5>
<p>Including information on habitats (biological community, biotope, or habitat type) is possible and encouraged with the use of Event Core. However, beware the unconstrained nature of the terms <code>measurementTypeID</code>, <code>measurementValueID</code>, and <code>measurementUnitID</code> which can lead to inconsistently documented habitat measurements within the Darwin Core Archive standard. To ensure this data is more easily discoverable, understood or usable, refer to <a href="other_data_types.html#habitat-data">Examples: habitat data</a> and/or <a href="https://emodnet.ec.europa.eu/en/reports?field_emodnet_lot_value%5B%5D=seabed_habitats#h3298bcd0a15741a8a0ac1c8b4576f7c5">Duncan et al.&nbsp;(2021)</a> for use case examples and more details.</p>
</section>
<section id="recommended-reading" class="level5" data-number="2.4.2.2.4">
<h5 data-number="2.4.2.2.4" class="anchored" data-anchor-id="recommended-reading"><span class="header-section-number">2.4.2.2.4</span> Recommended reading</h5>
<ul>
<li><a href="https://bdj.pensoft.net/articles.php?id=10989">De Pooter et al.&nbsp;2017</a>. Toward a new data standard for combined marine biological and environmental datasets - expanding OBIS beyond species occurrences. Biodiversity Data Journal 5: e10989. hdl.handle.net/10.3897/BDJ.5.e10989</li>
<li><a href="https://emodnet.ec.europa.eu/en/reports?field_emodnet_lot_value%5B%5D=seabed_habitats#h3298bcd0a15741a8a0ac1c8b4576f7c5">Duncan et al.&nbsp;(2021)</a>. A standard approach to structuring classified habitat data using the Darwin Core Extended Measurement or Fact Extension. EMODnet report. <em>(Note you must refine search to Technical Reports from 2021 to identify Duncan et al.’s report)</em></li>
</ul>
</section>
</section>
</section>
<section id="relational-databases-the-underlying-framework-of-obis" class="level3" data-number="2.4.3">
<h3 data-number="2.4.3" class="anchored" data-anchor-id="relational-databases-the-underlying-framework-of-obis"><span class="header-section-number">2.4.3</span> Relational databases: the underlying framework of OBIS</h3>
<p>If you are not familiar with relational databases, it can be difficult to understand the underlying framework OBIS relies on. This section will help you understand relational databases, how they relate to OBIS, the data you will format for OBIS, and the data you may download from OBIS.</p>
<p>Why do we use relational databases in the first place? You are probably familiar with flat databases which contain all data in one table - this is likely how your own data are formatted. Relational databases instead consist of multiple data tables that each contain <em>related</em> information. When all this information is presented in one table, the table becomes larger, very complicated, and the likelihood of data duplication increases. Relational databases seek to simplify complexities and <a href="relational_db.html#how-to-avoid-redundancy">reduce redundancy</a> by allowing information to be self-contained, but linked to each other.</p>
<p>You can think of a relational database as separate Excel sheets or data tables that are related to each other. One data table could be a “core” table, whereas others are “extensions”. Sometimes the relationships between core and extension tables are hierarchical, but this is not always the case. There is, however, always a <em>relationship</em> linking core and extension tables.</p>
<p>Let’s review core and extension tables and how we use them for OBIS.</p>
<p>Core tables contain information that is applicable to <strong>all</strong> extension tables, and extension tables contain more information about the records within the Core table. Each table, whether core or extension, contains records and attributes. Each row is a record (e.g., a sampling event, a species’ occurrence), whereas each column is an attribute (e.g., a date, a measurement).</p>
<p>Records between tables are linked to each other by the use of <em>identifiers</em>. A description of measurements pertaining to a record in an Extension table will have the same identifier as the record it is describing in the Core table. By using identifiers to link records, we reduce data repetition, see <a href="relational_db.html#how-to-avoid-redundancy">below</a> for examples. In the Darwin Core format that OBIS uses, the core table is either <a href="https://manual.obis.org/data_format.html#when-to-use-event-core">Event</a> or <a href="https://manual.obis.org/data_format.html#when-to-use-occurrence-core">Occurrence</a>, and datasets can have <a href="formatting.html#extensions-accepted-by-OBIS">one, none, or more</a> extension tables. Further explanation of data formatting in OBIS is covered in the <a href="formatting.html">Data Formatting section</a> of the OBIS manual.</p>
<p>Let’s review an example to fully understand how relational databases work. We will look at a simple relational database used by a fictional country that tracks student performance in three different courses between three schools. Rather than trying to contain information about each school, course, and student performance in one place, this information is split into three separate tables. We see that the pink table gives us information about each school - its name, and the district it belongs to. Each school also has a schoolID, an identifier linking to the blue table where we can see student performance (course mean) in each course, the class size, and year. You will notice that the course mean and class size are bundled under columns called measurementType and measurementValue. These are similar to the eMoF vocabularies and are integral to reducing repeated data, especially when one dataset has reoccurring information. Finally we see that the courseID in the blue table links to the yellow one with the courseID identifier, giving us information about each course.</p>
<p>A fourth table could easily be created to track total school population size through time. In contrast, if this information was only presented in the pink Schools in Country table, the school information would be duplicated as you add rows for each year. In this way, you can easily see how useful relational databases are. Of course, this is a simplified example but it demonstrates how related tables can be linked by identifiers to reduce table complexity and data replication.</p>
<p>We elaborate on how this structure is applied within OBIS <a href="formatting.html#dataset-structure">here</a>.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/relationalDB-updated.png" class="img-fluid figure-img" style="width:80.0%"></p>
<figcaption class="figure-caption"><em>An example of how a relational database works. Three tables show the (1) student performance (blue table) in (2) different schools (pink table) in a fictional country, and (3) the names of the courses (yellow table). Information between each table is linked by the use of identifiers, indicated by the arrows</em></figcaption>
</figure>
</div>
<p>Note that when OBIS harvests data, datasets are flattened - i.e., all separate data tables are combined into one. This is the kind of file you will receive when you <a href="access.html">download data from OBIS</a>. The reason for this is that querying relational databases significantly reduces computational time, as opposed to querying a flat database. Relational databases also facilitate requests for subsets that meet particular criteria - e.g., all data from Norway for one species above a certain depth.</p>
<section id="how-to-avoid-redundancy" class="level4" data-number="2.4.3.1">
<h4 data-number="2.4.3.1" class="anchored" data-anchor-id="how-to-avoid-redundancy"><span class="header-section-number">2.4.3.1</span> How to avoid redundancy</h4>
<p>Avoiding redundancy and data duplication within your dataset is built into the OBIS data structure. Utilizing the ENV-DATA approach, which delineates relationships between the core table and extension tables, we can limit the repetition of data.</p>
<p>For example, let us consider the dates of a ship cruise where a series of bottom trawls were taken. The sampling information (e.g., date range, equipment used, etc.) for each species collected in these trawls is the same. Because of this, we know we are dealing with unique sampling events and thus we will use <a href="format_event.html">Event core</a>. So, our Event core table will contain all information related to the sampling events (e.g., date, location). Then, information pertaining to each collected species (e.g., abundance, biomass, sampling methods, etc.) will be placed in an extension, the (Extended)MeasurementOrFact table. Here, each measurement for each species and sample will occur on a separate record. These records will be linked to the correct sampling event in the Event core by an identifier - the eventID. If we were to put this data in one file, the fields related to date and location (e.g., eventDate, decimalLongitude, decimalLatitude, etc.) would be repeated for each species.</p>
<p>Let’s consider another example. If you took one temperature measurement from the water column where you took your sample, each species found in that sample would have the <strong>same</strong> temperature measurement. By linking such measurements to the <em>event</em> instead of each <em>occurrence</em>, we are able to reduce the amount of data being repeated.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/OBIS-sampling-schema-example-update.png" class="img-fluid figure-img" style="width:80.0%"></p>
<figcaption class="figure-caption"><em>Example of how the sample data is distributed to Core and Extension tables, and how these tables are connected in OBIS</em></figcaption>
</figure>
</div>
<p>An advantage of structuring data this way is that if any mistakes are made, you only need to correct it once! So you can see that using relational event structures (when applicable) in combination with extension files can really simplify and reduce the number of times data are repeated.</p>
<p><strong>Caveat:</strong> However we would like to note that in some cases, data duplication may occur due to the star schema structure. For example, when publishing DNA-derived data, <a href="https://docs.gbif.org/publishing-dna-derived-data/1.0/en/#data-packaging-and-mapping">Occurrence core will have to be used</a>, which necessitates the repetition of event data for each occurrence record.</p>
</section>
</section>
<section id="ecological-metadata-language" class="level3" data-number="2.4.4">
<h3 data-number="2.4.4" class="anchored" data-anchor-id="ecological-metadata-language"><span class="header-section-number">2.4.4</span> Ecological Metadata Language</h3>
<p>OBIS (and GBIF) uses the Ecological Metadata Language (EML) as its metadata standard, which is specifically developed for the earth, environmental and ecological sciences. It is based on prior work done by the Ecological Society of America and associated efforts. EML is implemented as XML. See more information on <a href="https://eml.ecoinformatics.org/">EML</a>.</p>
<p>OBIS uses the <a href="http://rs.gbif.org/schema/eml-gbif-profile/1.1/eml-gbif-profile.xsd">GBIF EML profile (version 1.1)</a>. In case data providers use ISO19115/ISO19139, there is a mapping available <a href="http://rs.gbif.org/schema/eml-gbif-profile/1.1/eml2iso19139.xsl">here</a>.</p>
<p>For OBIS, the following 4 terms are the bare minimum required: <code>Title</code>, <code>Citation</code>, <code>Contact</code> and <code>Abstract</code>. Below is an overview of all the EML terms used to describe datasets:</p>
<ul>
<li><p><code>title [xml:lang="..."]</code>: A good descriptive <code>title</code> is indispensable and can provide the user with valuable information, making the discovery of data easier. Multiple titles may be provided, particularly when trying to express the title in more than one language (use the “xml:lang” attribute to indicate the language if not English/en).</p></li>
<li><p><code>creator</code> ; <code>metadataProvider</code> ; <code>associatedParty</code> ; <code>contact</code> : These are the people and organizations responsible for the dataset resource, either as the creator, the metadata provider, contact person or any other association. The following details can be provided:</p>
<ul>
<li><code>individualName</code>
<ul>
<li><code>givenName</code></li>
<li><code>surName</code></li>
</ul></li>
<li><code>organizationName</code>: Name of the institution.</li>
<li><code>positionName</code>: to be used as alternative to persons names (leave <code>individualName</code> blank and use <code>positionName</code> instead e.g.&nbsp;data manager).</li>
<li><code>address</code>
<ul>
<li><code>deliveryPoint</code></li>
<li><code>city</code></li>
<li><code>administrativeArea</code></li>
<li><code>postalCode</code></li>
<li><code>country</code></li>
</ul></li>
<li><code>phone</code><br>
</li>
<li><code>electronicMailAddress</code></li>
<li><code>onlineUrl</code> : personal website</li>
<li><code>role</code>: used with <code>associatedParty</code> to indicate the role of the associated person or organization.</li>
<li><code>userID</code>: e.g.&nbsp;ORCID.
<ul>
<li><code>directory</code></li>
</ul></li>
</ul></li>
<li><p><code>pubDate</code>: The date that the resource was published. Use ISO 8601.</p></li>
<li><p><code>language</code>: The language in which the resource (not the metadata document) is written. Use ISO language code.</p></li>
<li><p><code>abstract</code> : Brief description of the data resource.</p>
<ul>
<li><code>para</code></li>
</ul></li>
<li><p><code>keywordSet</code></p>
<ul>
<li><code>keyword</code> : Note only one keyword per keyword field is allowed.</li>
<li><code>keywordThesaurus</code> : e.g.&nbsp;ASFA</li>
</ul></li>
<li><p><code>additionalInfo</code> : OBIS checks this EML field for harvesting. It should contain <em>marine, harvested by iOBIS</em>.</p>
<ul>
<li><code>para</code></li>
</ul></li>
<li><p><code>coverage</code></p>
<ul>
<li><code>geographicCoverage</code>
<ul>
<li><code>geographicDescription</code>: a short text description of the area. E.g. the river mounth of the Scheldt Estuary.</li>
<li><code>boundingCoordinates</code>
<ul>
<li><code>westBoundingCoordinate</code></li>
<li><code>eastBoundingCoordinate</code></li>
<li><code>northBoundingCoordinate</code></li>
<li><code>southBoundingCoordinate</code></li>
</ul></li>
</ul></li>
<li><code>temporalCoverage</code> : Use ISO 8601
<ul>
<li><code>singleDateTime</code></li>
<li><code>rangeOfDates</code>
<ul>
<li><code>beginDate</code>
<ul>
<li><code>calendarDate</code></li>
</ul></li>
<li><code>endDate</code>
<ul>
<li><code>calendarDate</code></li>
</ul></li>
</ul></li>
</ul></li>
<li><code>taxonomicCoverage</code>: taxonomic information about the dataset. It can include a species list.
<ul>
<li><code>generalTaxonomicCoverage</code></li>
<li><code>taxonomicClassification</code>
<ul>
<li><code>taxonRankName</code></li>
<li><code>taxonRankValue</code></li>
<li><code>commonName</code></li>
</ul></li>
</ul></li>
</ul></li>
<li><p><code>intellectualRights</code>: Statement about IPR, Copyright or various Property Rights. Also read the <a href="policy.html">guidelines on the sharing and use of data in OBIS</a>.</p>
<ul>
<li><code>para</code></li>
</ul></li>
<li><p><code>purpose</code>: A description of the purpose of this dataset.</p>
<ul>
<li><code>para</code></li>
</ul></li>
<li><p><code>methods</code></p>
<ul>
<li><code>methodStep</code>: Descriptions of procedures, relevant literature, software, instrumentation, source data and any quality control measures taken.</li>
<li><code>sampling</code>: Description of sampling procedures including the geographic, temporal and taxonomic coverage of the study.</li>
<li><code>studyExtent</code>: Description of the specific sampling area, the sampling frequency (temporal boundaries, frequency of occurrence), and groups of living organisms sampled (taxonomic coverage).</li>
<li><code>samplingDescription</code>: Description of sampling procedures, similar to the one found in the methods section of a journal article.
<ul>
<li><code>para</code></li>
</ul></li>
<li><code>qualityControl</code>: Description of actions taken to either control or assess the quality of data resulting from the associated method step.</li>
</ul></li>
<li><p><code>project</code></p>
<ul>
<li><code>title</code></li>
<li><code>identifier</code></li>
<li><code>personnel</code>: The personnel field is used to document people involved in a research project by providing contact information and their role in the project.</li>
<li><code>description</code></li>
<li><code>funding</code>: The funding field is used to provide information about funding sources for the project such as: grant and contract numbers; names and addresses of funding sources.
<ul>
<li><code>para</code></li>
</ul></li>
<li><code>studyAreaDescription</code></li>
<li><code>designDescription</code>: The description of research design.</li>
</ul></li>
<li><p><code>maintenance</code></p>
<ul>
<li><code>description</code>
<ul>
<li><code>para</code></li>
</ul></li>
<li><code>maintenanceUpdateFrequency</code></li>
</ul></li>
<li><p><code>additionalMetadata</code></p>
<ul>
<li><code>metadata</code>
<ul>
<li><code>dateStamp</code>: The dateTime the metadata document was created or modified (ISO 8601).</li>
<li><code>metadataLanguage</code>: The language in which the metadata document (as opposed to the resource being described by the metadata) is written</li>
<li><code>hierarchyLevel</code>
<ul>
<li><code>citation</code> : A single citation for use when citing the dataset. The IPT can also auto-generate a citation based on the metadata (people, title, organization, onlineURL, DOI etc).</li>
<li><code>bibliography</code>: A list of citations that form a bibliography on literature related / used in the dataset</li>
<li><code>resourceLogoUrl</code>: URL of the logo associated with a dataset.</li>
<li><code>parentCollectionIdentifier</code></li>
<li><code>collectionIdentifier</code></li>
<li><code>formationPeriod</code>: Text description of the time period during which the collection was assembled. E.g., “Victorian”, or “1922 - 1932”, or “c.&nbsp;1750”.</li>
<li><code>livingTimePeriod</code>: Time period during which biological material was alive (for palaeontological collections).</li>
<li><code>specimenPreservationMethod</code></li>
<li><code>physical</code>
<ul>
<li><code>objectName</code></li>
<li><code>characterEncoding</code></li>
<li><code>dataFormat</code>
<ul>
<li><code>externallyDefinedFormat</code></li>
<li><code>formatName</code></li>
</ul></li>
<li><code>distribution</code>: URL links
<ul>
<li><code>online</code>
<ul>
<li><code>url function="download"</code></li>
<li><code>url function="information"</code></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
<li><p><code>alternateIdentifier</code>: It is a Universally Unique Identifier (UUID) for the EML document and not for the dataset. This term is optional.</p></li>
</ul>
<section id="metadata-sections" class="level4" data-number="2.4.4.1">
<h4 data-number="2.4.4.1" class="anchored" data-anchor-id="metadata-sections"><span class="header-section-number">2.4.4.1</span> Metadata Sections</h4>
<p>There are several categories/pages for metadata you must provide, which includes basic information about the:</p>
<ul>
<li>Dataset and data provider</li>
<li>Geographic/taxonomic/temporal coverage</li>
<li>Keywords</li>
<li>Hosting institution information</li>
<li>Information regarding associated project(s)</li>
<li>Sampling methods</li>
<li>How to cite the dataset</li>
<li>Museum collection (if applicable)</li>
<li>Other external links (e.g.&nbsp;a homepage) or additional metadata</li>
</ul>
<p>We review each of these sections below.</p>
<section id="title" class="level5" data-number="2.4.4.1.1">
<h5 data-number="2.4.4.1.1" class="anchored" data-anchor-id="title"><span class="header-section-number">2.4.4.1.1</span> Title</h5>
<p>The IPT requires you to provide a <em>Shortname</em>. Shortnames serve as an identifier for the resource within the IPT installation (so should be unique within your IPT), and will be used as a parameter in the URL to access the resource via the Internet. Please use only alphanumeric characters, hyphens, or underscores. E.g. <em>largenet_im</em> in <a href="http://ipt.vliz.be/eurobis/resource?r=largenet_im">http://ipt.vliz.be/eurobis/resource?r=largenet_im</a>. After creating a new dataset resource, the field title will be filled out with the short name you provided earlier. Please make sure you provide a dataset title following the guidelines below.</p>
<p>Dataset titles provided to OBIS node managers are often very cryptic, such as an acronym, and often only understandable by the data provider. However, to increase the discoverability and be useful for a larger audience, the dataset title should be as descriptive and complete as possible. OBIS recommends titles to contain information about the taxonomic, geographic and temporal coverage. If the dataset title does not meet these criteria and you believe the title should be changed, then contact the data provider with a suggestion or ask for a more descriptive title. If the dataset has already been published (made publicly available) - and therefore known by that title elsewhere, then the same title should be kept (even if it would not meet the proposed guidelines)! Changing the title of an already published dataset cannot be done, as this will generate confusion and possible duplicates in systems like OBIS or GBIF in a later stage.</p>
<p>The acronym or working title could still be documented in the metadata, so there is no confusion about how the full title is linked to the originally provided acronym or working title.</p>
<blockquote class="blockquote">
<p><strong>Caution:</strong> Always consult the data provider when changing a dataset title to a more workable and descriptive version.</p>
</blockquote>
<table class="table">
<colgroup>
<col style="width: 43%">
<col style="width: 56%">
</colgroup>
<thead>
<tr class="header">
<th>Originally received title</th>
<th>Title Recommended by Node Manager</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>BIOCEAN</td>
<td>BIOCEAN database on deep sea benthic fauna</td>
</tr>
<tr class="even">
<td>Biomôr</td>
<td>Benthic data from the Southern Irish Sea from 1989-1991</td>
</tr>
<tr class="odd">
<td>Kyklades</td>
<td>Zoobenthos of the Kyklades (Aegean Sea)</td>
</tr>
<tr class="even">
<td>REPHY</td>
<td>Réseau de Surveillance phytoplanctonique</td>
</tr>
</tbody>
</table>
</section>
<section id="abstract" class="level5" data-number="2.4.4.1.2">
<h5 data-number="2.4.4.1.2" class="anchored" data-anchor-id="abstract"><span class="header-section-number">2.4.4.1.2</span> Abstract</h5>
<p>The abstract or description of a dataset provides basic information on the content of the dataset. The information in the abstract should improve understanding and interpretation of the data. It is recommended that the description indicates whether the dataset is a subset of a larger dataset and – if so – provide a link to the parent metadata and/or dataset.</p>
<p>If the data provider or OBIS node require bi- or multilingual entries for the description (e.g.&nbsp;due to national obligations) then the following procedure can be followed:</p>
<ul>
<li>Indicate English as metadata language</li>
<li>Enter the English description first</li>
<li>Type a slash (/)</li>
<li>Enter the description in the second language</li>
</ul>
<p><em>Example:</em> The Louis-Marie herbarium grants a priority to the Arctic-alpine, subarctic and boreal species from the province of Quebec and the northern hemisphere. This dataset is mainly populated with specimens from the province of Quebec. / L’Herbier Louis-Marie accorde une priorité aux espèces arctiques-alpines, subarctiques et boréales du Québec, du Canada et de l’hémisphère nord. Ce jeu présente principalement des spécimens provenant du Québec.</p>
</section>
<section id="people-and-organizations" class="level5" data-number="2.4.4.1.3">
<h5 data-number="2.4.4.1.3" class="anchored" data-anchor-id="people-and-organizations"><span class="header-section-number">2.4.4.1.3</span> People and Organizations</h5>
<p>The EML has several possible roles/functions to describe a contact, creator, metadata provider and associated party.</p>
<p>The <code>contact</code> is the person or organization that curates the resource and who should be contacted to get more information or to whom questions with the resource or data should be addressed. Although a number of fields are not required, we strongly recommend providing as much information as possible, and in particular the email address. This will also be the contact information that appears on the OBIS metadata pages.</p>
<p>The <code>creator</code> is the person or organization responsible for the original creation of the resource content. When there are multiple creators, the one that bears the greatest responsibility is the resource creator, and other people can be added as associated parties with a role such as ‘originator’, ‘content provider’, ‘principal investigator’, etc.</p>
<p>Possible functions/roles:</p>
<ul>
<li>Originator (person/organization that originally gathered/prepared the dataset)</li>
<li>Content provider (principal person/organization that contributed content to the dataset)</li>
</ul>
<p>If the resource contact and the resource creator are identical, the IPT allows you to easily copy the information.</p>
<p>The <code>metadata provider</code> is the person or organization responsible for producing the resource metadata. If the metadata are provided by the original data provider, then his/her contact details should be filled in. If no metadata are available (e.g.&nbsp;for historical datasets, with no contact person), then the metadata can be completed by e.g.&nbsp;the OBIS node manager and the OBIS node manager becomes the metadata provider.</p>
<p>The <code>Associated Parties</code> contains information about one or more people or organizations associated with the resource in addition to those already covered on the IPT Basic Metadata page. For example, if there would be multiple contact persons or metadata creators, they can be added in this IPT section. The principal contact/creator should, however, be added in the IPT Basic Metadata section, not the <code>Associated Parties</code> section. It is recommended to complete this section together with the IPT Basic Metadata page, to avoid confusion or overlap in added information.</p>
<p>Possible functions/roles for associated parties are:</p>
<ul>
<li>Custodian steward (person/organization responsible for/takes care of the dataset paper)</li>
<li>Owner (person/organization that owns the data – may or may not be the custodian)</li>
<li>Point of contact (person/organization to contact for further information on the dataset)</li>
<li>principal investigator (primary scientific contact associated with the dataset)</li>
</ul>
<p><em>Notes:</em></p>
<p>The owner of a dataset will, in most cases, be an institute, and not an individual person. Although the fields ‘last name’, and ‘position’ are indicated as mandatory fields, it is possible to just add the institute name in the ‘last name’ field for the role ‘owner’.</p>
<p>The contact persons in the metadata (contact, creator, metadata creator) are used in the dataset citation (auto-generation) and those added as ‘associated parties’ are not included as “co-authors”.</p>
</section>
<section id="license-and-ip-rights" class="level5" data-number="2.4.4.1.4">
<h5 data-number="2.4.4.1.4" class="anchored" data-anchor-id="license-and-ip-rights"><span class="header-section-number">2.4.4.1.4</span> License and IP Rights</h5>
<p>OBIS has published its guidelines on the sharing and use of data <a href="policy.html">here</a>. The recommended licenses for datasets published in OBIS are the Creative Commons Licenses (CC-0, CC-BY, CC-BY-NC), of which CC-0 is the most preferred and CC-BY-NC is least preferred. A Creative Commons license means:</p>
<ul>
<li>You are free:
<ul>
<li>to share =&gt; to copy, distribute and use the database</li>
<li>to create =&gt; to produce works from the database</li>
<li>to adapt =&gt; to modify, transform and build upon the database</li>
</ul></li>
</ul>
<p><strong>In case of CC-0:</strong> <code>public domain</code>: CC-0 is the preferred option identified by the OBIS steering group. You waive any copyright you might have over the data(set) and dedicate it to the public domain. You cannot be held liable for any (mis)use of the data either. Although CC-0 doesn’t legally require users of the data to cite the source, it does not take away the moral responsibility to give attribution, as is common in scientific research. A good blog on why using CC-0 can be found <a href="https://community.canadensys.net/2012/why-we-should-publish-our-data-under-cc0">here</a>.</p>
<p><strong>In case of CC-BY:</strong> <code>Attribution</code>: You must attribute any public use of the database, or works produced from the database, in the manner specified in the license. For any use or redistribution of the database, or works produced from it, you must make clear to others the license of the database and keep intact any notices on the original database.</p>
<p><strong>In case of CC-BY-NC:</strong> <code>non-commercial</code>: like CC-BY but commercial use is not allowed. This licence can be problematic when the data is re-used in scientific journals.</p>
</section>
<section id="coverage" class="level5" data-number="2.4.4.1.5">
<h5 data-number="2.4.4.1.5" class="anchored" data-anchor-id="coverage"><span class="header-section-number">2.4.4.1.5</span> Coverage</h5>
<section id="geographic-coverage" class="level6" data-number="2.4.4.1.5.1">
<h6 data-number="2.4.4.1.5.1" class="anchored" data-anchor-id="geographic-coverage"><span class="header-section-number">2.4.4.1.5.1</span> Geographic Coverage</h6>
<p>The IPT allows you to enter the geographic coverage by dragging the markers on the given map or by filling in the coordinates of the bounding box. In the description field, a more elaborate text can be provided to describe the spatial coverage indicating the larger geographical area where the samples were collected. For the latter, the sampling locations can be plotted on a map and – by making use of a Gazetteer – the wider geographical area can be derived: e.g.&nbsp;the relevant Exclusive Economic Zone (EEZ), IHO, FAO fishing area, Large Marine Ecosystem (LME), Marine Ecoregions of the World (MEOW), etc. The <a href="http://www.marineregions.org/">Marine Regions’ Gazetteer</a> might prove to be a useful online tool to define the most relevant sea area(s). There are also <a href="http://www.lifewatch.be/data-services/">LifeWatch Geographical Services</a> that translate geographical positions to these wider geographical areas.</p>
<p>The information given in this section can also help the OBIS node manager in geographic quality control. If the geographic coverage in the EML e.g.&nbsp;is “North Sea”, but a number of data points are outside of this scope, then this may indicate errors, and should be checked with the data provider.</p>
<p>If the dataset covers multiple areas (e.g.&nbsp;samples from the North Sea and the Mediterranean Sea), then this should clearly be mentioned in the <code>geographicDescription</code> field. Note that the IPT only allows one bounding box, and you have to uncheck the “Set global coverage” box to change box bounds.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ipt-ss13-meta-geo.png" class="img-fluid figure-img" style="width:60.0%"></p>
<figcaption class="figure-caption">Screenshot of the Geographical Coverage section of the metadata, emphasizing how to change the bounds of the coverage box in the map.</figcaption>
</figure>
</div>
</section>
<section id="taxonomic-coverage" class="level6" data-number="2.4.4.1.5.2">
<h6 data-number="2.4.4.1.5.2" class="anchored" data-anchor-id="taxonomic-coverage"><span class="header-section-number">2.4.4.1.5.2</span> Taxonomic Coverage</h6>
<p>This section can capture two things:</p>
<ol type="1">
<li>A description of the range of taxa that are addressed in the data set. OBIS recommends to only add the higher classification (Kingdom, Class or Order) of the involved groups (e.g.&nbsp;Bivalvia, Cetacea, Aves, Ophiuroidea…). You can easily draw a list of higher taxonomic ranks from the WoRMS taxon match service (or ask the data provider). The taxonomic coverage is not a mandatory field, but the information stored here can be very useful as background information. The description can also contain common names, such as e.g.&nbsp;benthic foraminifera or mussels.</li>
<li>An overview of all the involved taxa (not recommended, as all the taxa are already listed in the dataset).</li>
</ol>
<blockquote class="blockquote">
<p><em>Note:</em> OBIS also recommends to add information on the (higher) taxonomic groups in the (descriptive) dataset title and abstract.</p>
</blockquote>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ipt-ss14-meta-taxa.png" class="img-fluid figure-img" style="width:70.0%"></p>
<figcaption class="figure-caption">Example of the Taxonomic Coverage section of the metadata</figcaption>
</figure>
</div>
</section>
<section id="temporal-coverage" class="level6" data-number="2.4.4.1.5.3">
<h6 data-number="2.4.4.1.5.3" class="anchored" data-anchor-id="temporal-coverage"><span class="header-section-number">2.4.4.1.5.3</span> Temporal Coverage</h6>
<p>The temporal coverage will be a date range, which can easily be documented. If it is a single date, the start and end date will be the same. The information added here can be used as a quality check for the actual dates in the datasets.</p>
<p>You can also document the Formation Period or the Living Time Period in this section for specimens that may not have been alive during the collection period, or to indicate the time during which the collection occurred.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ipt-ss15-meta-time.png" class="img-fluid figure-img" style="width:70.0%"></p>
<figcaption class="figure-caption">Example of the Temporal Coverage section of the metadata</figcaption>
</figure>
</div>
</section>
</section>
<section id="keywords" class="level5" data-number="2.4.4.1.6">
<h5 data-number="2.4.4.1.6" class="anchored" data-anchor-id="keywords"><span class="header-section-number">2.4.4.1.6</span> Keywords</h5>
<p>Relevant keywords facilitate the discovery of a dataset. An indication of the represented functional groups can help in a general search (e.g.&nbsp;plankton, benthos, zooplankton, phytoplankton, macrobenthos, meiobenthos …). Assigned keywords can be related to taxonomy, habitat, geography or relevant keywords extracted from thesauri such as the <a href="https://vocabularyserver.com/asfa/">ASFA thesaurus</a>, the <a href="http://www.cabi.org/cabthesaurus/">CAB thesaurus</a> or <a href="https://www.earthdata.nasa.gov/learn/find-data/idn/gcmd-keywords">GCMD keywords</a>.</p>
<p>As taxonomy and geography are already covered in previous sections, there is no need to repeat related keywords here. Please consult your data provider which (relevant) keywords can be assigned.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ipt-ss16-meta-keyword.png" class="img-fluid figure-img" style="width:70.0%"></p>
<figcaption class="figure-caption">Example of the Keywords section of the metadata, showing input for a marine fishes dataset</figcaption>
</figure>
</div>
</section>
<section id="project" class="level5" data-number="2.4.4.1.7">
<h5 data-number="2.4.4.1.7" class="anchored" data-anchor-id="project"><span class="header-section-number">2.4.4.1.7</span> Project</h5>
<p>If the dataset in this resource is produced under a certain project, the metadata on this project can be documented here. Part of the information entered here, can partly overlap with information given in other sections of the metadata (e.g.&nbsp;study area description can have lot of parallel with the geographic coverage section). Personnel involved in the project can be documented or repeated here as well. This is not a problem.</p>
</section>
<section id="sampling-methods" class="level5" data-number="2.4.4.1.8">
<h5 data-number="2.4.4.1.8" class="anchored" data-anchor-id="sampling-methods"><span class="header-section-number">2.4.4.1.8</span> Sampling Methods</h5>
<p>The EML can contain descriptions of the sampling and data processing methods. Study extent can be documented here as well to report a more specific geographic area as well as the sampling frequency. Descriptions of sampling procedures, quality control, and steps (sample or data processing) can be given in the same way as the methods section of a scientific paper.</p>
<p>Note that OBIS best practice is to add sampling facts to the extended MeasurementorFact extension, linked to the sampling events in the Event core via eventID.</p>
</section>
<section id="citations" class="level5" data-number="2.4.4.1.9">
<h5 data-number="2.4.4.1.9" class="anchored" data-anchor-id="citations"><span class="header-section-number">2.4.4.1.9</span> Citations</h5>
<p>The dataset citation allows users to properly cite your dataset in further publications or other uses of the data. When users download datasets from the OBIS download function, a list of the dataset citations packaged with the data in a zipped file is provided.</p>
<p>A dataset citation is different from the data source citation (in case the data is digitized from a publication), and these references can be added to the additional metadata (see bibliography below). A dataset citation can have the same format of a journal article citation, and should include the authors (contact, creator, principle investigator, data managers, custodians, collectors…), the title of the dataset, the name of the data publisher (or custodian institute), and the access point URL to the resource.</p>
<p>GBIF’s IPT has an auto-generation - Turn On/Off - tool to let the IPT auto-generate the resource citation for you. The citation includes a version number, which is especially important for datasets that are continuously updated. The dataset citation can also include a Citation Identifier - a DOI, URI, or other persistent identifier that resolves to an online dataset web page.</p>
<p>The OBIS node data managers should try to implement a certain degree of format standardization for the dataset citations. The IPT provides an option to auto-generate a citation based on the EML and is formatted as follows: {dataset.authors} ({dataset.pubDate}) {dataset.title}. [Version {dataset.version}]. {organization.title}. {dataset.type} Dataset {dataset.doi}, {dataset.url}</p>
</section>
<section id="bibliography" class="level5" data-number="2.4.4.1.10">
<h5 data-number="2.4.4.1.10" class="anchored" data-anchor-id="bibliography"><span class="header-section-number">2.4.4.1.10</span> Bibliography</h5>
<p>The EML can include the citation of the publications that are related to the described dataset. They can describe the dataset, be based on the dataset or be used in this dataset. Publications can be scientific papers, reports, PhD or master theses. If available, the citation should include the DOI at the end.</p>
<p>This overview will contribute to a better understanding of the data as these publications can hold important additional information on the data and how they were acquired.</p>
</section>
<section id="collection-data" class="level5" data-number="2.4.4.1.11">
<h5 data-number="2.4.4.1.11" class="anchored" data-anchor-id="collection-data"><span class="header-section-number">2.4.4.1.11</span> Collection Data</h5>
<p>This IPT section should only be filled out if there are specimens held in a museum. If relevant, it is strongly recommended that this information is supplied by the data provider or left blank. The collection name, specimen preservation method, and curatorial units should be provided, as applicable.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ipt-ss17-meta-collection.png" class="img-fluid figure-img" style="width:70.0%"></p>
<figcaption class="figure-caption">Screenshot of the Collection Data page showing what information can be provided for museum specimens</figcaption>
</figure>
</div>
</section>
<section id="external-links" class="level5" data-number="2.4.4.1.12">
<h5 data-number="2.4.4.1.12" class="anchored" data-anchor-id="external-links"><span class="header-section-number">2.4.4.1.12</span> External Links</h5>
<p>This section can include URLs to the resource homepage, to download or find additional information. You can also provide links to your resource if it is hosted elsewhere in different formats.</p>
<p>Links to the online dataset on the OBIS website can be added once the data is available there. For these OBIS links, the required fields should be completed as follows:</p>
<ul>
<li>Name: online dataset</li>
<li>Character set: UTF-8</li>
<li>Data format: html</li>
</ul>
<p>If other links are added, then the data format for web-based data is ‘html’. If the link refers to a file, the data format of the file will need to be added (e.g.&nbsp;.xlsx, .pdf …). The character set for all Darwin Core files is UTF-8, whereas for other web pages this can vary, so you may need to confirm.</p>
</section>
<section id="additional-metadata" class="level5" data-number="2.4.4.1.13">
<h5 data-number="2.4.4.1.13" class="anchored" data-anchor-id="additional-metadata"><span class="header-section-number">2.4.4.1.13</span> Additional Metadata</h5>
<p>Any remaining information that could not be catalogued under any of the other metadata, can be mentioned here. This may include logos, purpose of the dataset, a description of how the dataset will be maintained, etc.</p>
</section>
</section>
</section>
</section>
<section id="obis-nodes" class="level2" data-number="2.5">
<h2 data-number="2.5" class="anchored" data-anchor-id="obis-nodes"><span class="header-section-number">2.5</span> OBIS nodes</h2>
<p><em>Note the OBIS node TOR and system architecture is currently under review and will be updated after the 2023 Steering Group meeting. The information below may change.</em></p>
<p>OBIS Nodes are either national projects, programmes, institutes, or organizations, National Ocean Data Centers or regional or international projects, programmes and institutions or organizations that carry out data management functions.</p>
<p>OBIS nodes are responsible for <strong>representing all aspects of OBIS within a particular region or taxonomic domain</strong>. Additional responsibilities include:</p>
<ul>
<li>Establishing relationships with key data providers within their geographical (or taxonomic) area of responsibility</li>
<li>Bringing data and corresponding metadata into the global database to be shared with the OBIS community</li>
<li>Responsibility for <strong>all aspects of the data</strong></li>
<li>Gaining permission to providing access to the data</li>
<li>Ensuring a certain level of data quality</li>
<li>Transfer of these datasets to the global OBIS database</li>
<li>Provide support for the full implementation of OBIS worldwide by serving on the IODE Steering Group for OBIS and any relevant Task Teams or ad hoc project teams</li>
<li>Each node may also maintain a data presence on the Internet representing their specific area of responsibility</li>
</ul>
<section id="terms-of-reference-of-obis-nodes" class="level3" data-number="2.5.1">
<h3 data-number="2.5.1" class="anchored" data-anchor-id="terms-of-reference-of-obis-nodes"><span class="header-section-number">2.5.1</span> Terms of Reference of OBIS nodes</h3>
<p><strong>Data Responsibilities</strong></p>
<ul>
<li>Receiving or harvesting marine biodiversity data (and metadata) from national, regional, and international programs, and the scientific community at large, and from Tier III nodes by Tier II nodes, and from Tier II nodes by Tier I nodes</li>
<li>Perform data validation (using standards, tools, and best practices), as described in the OBIS manual (Tier II)</li>
<li>Reporting the results of quality control directly to data collectors/originator (or Tier III node) as part of the quality assurance activity</li>
<li>Making data (and metadata) available to OBIS using agreed upon standards and formats which are described in the OBIS Manual (Tier II), making data available to Tier II nodes (Tier III)</li>
<li>Control data access, terms of use and sharing policies</li>
<li>Comply with the IOC/OBIS data policy for using and sharing OBIS data</li>
<li>Contribute to the development of standards and best practices in OBIS (recommended)</li>
<li>Contribute to the development of open-source tools in OBIS (recommended)</li>
<li>Ensuring the long-term preservation of the data, metadata and associated information required for correct interpretation of the data (including version-control) (recommended)</li>
<li>Build customized data portals (optional)</li>
</ul>
<p><strong>Administration Responsibilities</strong></p>
<ul>
<li>Become a member of the IODE steering group for OBIS, attend the SG-OBIS annual meeting and report on node activities</li>
<li>Provide indicators on up-time, responsiveness, and data processed by nodes and present a report to SG-OBIS</li>
<li>Customer support (data queries, analyses, feedback)</li>
<li>Outreach and Capacity Building (i.e., providing expertise, training and support in data management, technologies, standards and best practices)</li>
<li>Engage in stakeholder groups (recommended)</li>
</ul>
</section>
<section id="how-to-become-an-obis-node" class="level3" data-number="2.5.2">
<h3 data-number="2.5.2" class="anchored" data-anchor-id="how-to-become-an-obis-node"><span class="header-section-number">2.5.2</span> How to become an OBIS node</h3>
<p>OBIS nodes now operate under the IODE network as either National Oceanographic Data Centres (NODCs) or Associate Data Unites (ADUs). Prospective nodes are required to apply to the IODE for membership.</p>
<p>The procedure to become an OBIS node is as follows:</p>
<ul>
<li>If you are an existing NODC (within the IODE network) and the OBIS node activities fall under the activities of the NODC:
<ul>
<li>Send a letter expressing your interest to become an OBIS node (including contact information of the OBIS node manager, and geographical/thematic scope of your OBIS node)</li>
</ul></li>
<li>If you are not an existing NODC:
<ul>
<li>Email your <a href="http://iode.org/index.php?option=com_oe&amp;task=viewDocumentRecord&amp;docID=11793">application form</a> to become an IODE Associate Data Unit (ADU), with a specific role as OBIS node. Applications for ADU membership in OBIS shall be reviewed by the IODE Officers in consultation with the IODE Steering Group for OBIS.</li>
</ul></li>
</ul>
</section>
<section id="obis-node-health-status-check-and-transition-strategy" class="level3" data-number="2.5.3">
<h3 data-number="2.5.3" class="anchored" data-anchor-id="obis-node-health-status-check-and-transition-strategy"><span class="header-section-number">2.5.3</span> OBIS Node Health Status Check and Transition Strategy</h3>
<p>OBIS nodes should operate under IODE as either IODE/ADU or IODE/NODC. As such OBIS nodes are a member of the IODE network.</p>
<p>The IODE Steering Group (SG) for OBIS evaluates the health status of OBIS nodes at each annual SG meeting, and considers an OBIS node as <strong>inactive</strong> when it meets any of the following conditions:</p>
<ol type="1">
<li>The OBIS node manager recurrently fails to answer the communications from the project manager or the SG co-chairs in the last 12 months</li>
<li>The OBIS node manager or a representative fails to attend (personally or virtually) the last 2 SG meetings without any written reason</li>
<li>The OBIS node does not have an IPT</li>
<li>The OBIS node has an IPT, but it has not been running for the last 12 months</li>
<li>The datasets in the OBIS node’s IPT have been removed and not restored in the last 12 months (without any explanation)</li>
<li>The OBIS node has not provided new data for the last 2 years</li>
</ol>
<p>The OBIS Secretariat prepares a health status check report of each OBIS node based on the six items above and informs the OBIS node manager on their status 3 months before the SG meeting. At the SG meeting, the SG-OBIS co-chair will present the results of the OBIS nodes health status check report including a listing of the inactive OBIS nodes. The SG-OBIS members representing active OBIS Nodes will make one of the following decisions:</p>
<ol type="1">
<li>Request the inactive OBIS node to submit a plan with actions, deliverables and times to improve their performance, within 3 months, to the OBIS Secretariat. This plan is reviewed and accepted by the OBIS-Executive Committee Or</li>
<li>Provide a recommendation to the IOC Committee on IODE to remove the OBIS node from the IODE network.</li>
</ol>
<p>In either case, the OBIS Secretariat will inform the OBIS node manager of the SG-OBIS decision, with a copy to the IODE officers and the IODE national coordinator for data management of the country concerned.</p>
<p>The IODE Committee is requested to consider the recommendation from the OBIS Steering Group and it may either accept the recommendation or request the inactive OBIS node to submit an action plan (option 1).</p>
<p>When the inactive OBIS node is removed from the IODE network, the SG-OBIS will ask whether another OBIS node is interested in taking over the responsibilities of the removed OBIS node, until a new OBIS node in the country/region is established.</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./intro.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Introduction</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./formatting.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Dataset structure</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>